2019-10-22 21:19:13.202 +02:00 [Information] No database connection exists yet. Initializing new connection.
2019-10-22 21:19:13.316 +02:00 [Information] Initialize RepositoryInspectorJobScheduler Run().
2019-10-22 21:19:13.688 +02:00 [Information] No RepositoryInspectorJobs found. Setting up InitTimer "System.Timers.Timer" to run in 1 s.
2019-10-22 21:19:13.693 +02:00 [Information] Startup Application.
2019-10-22 21:19:13.698 +02:00 [Debug] Hosting starting
2019-10-22 21:19:13.723 +02:00 [Information] User profile is available. Using '"/home/morrismorrison/.aspnet/DataProtection-Keys"' as key repository; keys will not be encrypted at rest.
2019-10-22 21:19:13.727 +02:00 [Debug] Reading data from file '"/home/morrismorrison/.aspnet/DataProtection-Keys/key-082815ae-49f7-4105-802a-401777056be8.xml"'.
2019-10-22 21:19:13.742 +02:00 [Debug] Found key {082815ae-49f7-4105-802a-401777056be8}.
2019-10-22 21:19:13.753 +02:00 [Debug] Considering key {082815ae-49f7-4105-802a-401777056be8} with expiration date 2020-01-03 00:58:56Z as default key.
2019-10-22 21:19:13.762 +02:00 [Debug] Forwarded activator type request from "Microsoft.AspNetCore.DataProtection.AuthenticatedEncryption.ConfigurationModel.AuthenticatedEncryptorDescriptorDeserializer, Microsoft.AspNetCore.DataProtection, Version=3.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60" to "Microsoft.AspNetCore.DataProtection.AuthenticatedEncryption.ConfigurationModel.AuthenticatedEncryptorDescriptorDeserializer, Microsoft.AspNetCore.DataProtection, Culture=neutral, PublicKeyToken=adb9793829ddae60"
2019-10-22 21:19:13.763 +02:00 [Debug] Using managed symmetric algorithm '"System.Security.Cryptography.Aes"'.
2019-10-22 21:19:13.764 +02:00 [Debug] Using managed keyed hash algorithm '"System.Security.Cryptography.HMACSHA256"'.
2019-10-22 21:19:13.767 +02:00 [Debug] Using key {082815ae-49f7-4105-802a-401777056be8} as the default key.
2019-10-22 21:19:13.767 +02:00 [Debug] Key ring with default key {082815ae-49f7-4105-802a-401777056be8} was loaded during application startup.
2019-10-22 21:19:13.781 +02:00 [Debug] Compatibility switch "AllowInputFormatterExceptionMessages" in type "MvcJsonOptions" is using default value False
2019-10-22 21:19:13.791 +02:00 [Debug] Compatibility switch "AllowCombiningAuthorizeFilters" in type "MvcOptions" is using default value False
2019-10-22 21:19:13.791 +02:00 [Debug] Compatibility switch "AllowBindingHeaderValuesToNonStringModelTypes" in type "MvcOptions" is using default value False
2019-10-22 21:19:13.791 +02:00 [Debug] Compatibility switch "AllowValidatingTopLevelNodes" in type "MvcOptions" is using default value False
2019-10-22 21:19:13.791 +02:00 [Debug] Compatibility switch "InputFormatterExceptionPolicy" in type "MvcOptions" is using default value AllExceptions
2019-10-22 21:19:13.791 +02:00 [Debug] Compatibility switch "SuppressBindingUndefinedValueToEnumType" in type "MvcOptions" is using default value False
2019-10-22 21:19:13.792 +02:00 [Debug] Compatibility switch "EnableEndpointRouting" in type "MvcOptions" is using default value False
2019-10-22 21:19:13.792 +02:00 [Debug] Compatibility switch "MaxValidationDepth" in type "MvcOptions" is using default value null
2019-10-22 21:19:13.792 +02:00 [Debug] Compatibility switch "AllowShortCircuitingValidationWhenNoValidatorsArePresent" in type "MvcOptions" is using default value False
2019-10-22 21:19:13.803 +02:00 [Debug] Compatibility switch "SuppressMapClientErrors" in type "ApiBehaviorOptions" is using compatibility value True for version Version_2_0
2019-10-22 21:19:13.803 +02:00 [Debug] Compatibility switch "SuppressUseValidationProblemDetailsForInvalidModelStateResponses" in type "ApiBehaviorOptions" is using compatibility value True for version Version_2_0
2019-10-22 21:19:13.803 +02:00 [Debug] Compatibility switch "AllowInferringBindingSourceForCollectionTypesAsFromQuery" in type "ApiBehaviorOptions" is using compatibility value True for version Version_2_0
2019-10-22 21:19:13.806 +02:00 [Debug] Compatibility switch "AllowAreas" in type "RazorPagesOptions" is using default value False
2019-10-22 21:19:13.806 +02:00 [Debug] Compatibility switch "AllowMappingHeadRequestsToGetHandler" in type "RazorPagesOptions" is using default value False
2019-10-22 21:19:13.806 +02:00 [Debug] Compatibility switch "AllowDefaultHandlingForOptionsRequests" in type "RazorPagesOptions" is using default value False
2019-10-22 21:19:13.810 +02:00 [Debug] Compatibility switch "AllowRecompilingViewsOnFileChange" in type "RazorViewEngineOptions" is using explicitly configured value True
2019-10-22 21:19:13.825 +02:00 [Debug] Compatibility switch "SuppressTempDataAttributePrefix" in type "MvcViewOptions" is using default value False
2019-10-22 21:19:13.826 +02:00 [Debug] Compatibility switch "AllowRenderingMaxLengthAttribute" in type "MvcViewOptions" is using default value False
2019-10-22 21:19:13.973 +02:00 [Debug] Registered model binder providers, in the following order: ["Microsoft.AspNetCore.Mvc.ModelBinding.Binders.BinderTypeModelBinderProvider", "Microsoft.AspNetCore.Mvc.ModelBinding.Binders.ServicesModelBinderProvider", "Microsoft.AspNetCore.Mvc.ModelBinding.Binders.BodyModelBinderProvider", "Microsoft.AspNetCore.Mvc.ModelBinding.Binders.HeaderModelBinderProvider", "Microsoft.AspNetCore.Mvc.ModelBinding.Binders.FloatingPointTypeModelBinderProvider", "Microsoft.AspNetCore.Mvc.ModelBinding.Binders.EnumTypeModelBinderProvider", "Microsoft.AspNetCore.Mvc.ModelBinding.Binders.SimpleTypeModelBinderProvider", "Microsoft.AspNetCore.Mvc.ModelBinding.Binders.CancellationTokenModelBinderProvider", "Microsoft.AspNetCore.Mvc.ModelBinding.Binders.ByteArrayModelBinderProvider", "Microsoft.AspNetCore.Mvc.ModelBinding.Binders.FormFileModelBinderProvider", "Microsoft.AspNetCore.Mvc.ModelBinding.Binders.FormCollectionModelBinderProvider", "Microsoft.AspNetCore.Mvc.ModelBinding.Binders.KeyValuePairModelBinderProvider", "Microsoft.AspNetCore.Mvc.ModelBinding.Binders.DictionaryModelBinderProvider", "Microsoft.AspNetCore.Mvc.ModelBinding.Binders.ArrayModelBinderProvider", "Microsoft.AspNetCore.Mvc.ModelBinding.Binders.CollectionModelBinderProvider", "Microsoft.AspNetCore.Mvc.ModelBinding.Binders.ComplexTypeModelBinderProvider"]
2019-10-22 21:19:14.001 +02:00 [Information] Starting @angular/cli on port 46403...
2019-10-22 21:19:14.050 +02:00 [Debug] Failed to locate the development https certificate at 'null'.
2019-10-22 21:19:14.078 +02:00 [Debug] Using development certificate: "CN=localhost" (Thumbprint: "ACEE09FCF1B545DA9A02F815D344217C10B191C4")
2019-10-22 21:19:14.091 +02:00 [Debug] Hosting started
2019-10-22 21:19:14.091 +02:00 [Debug] Loaded hosting startup assembly "RepositoryNotifier"
2019-10-22 21:19:16.831 +02:00 [Information] > GithubPasswordNotifier@0.0.0 start /home/morrismorrison/Development/Projects/NET/RepositoryObserver/RepositoryNotifier/ClientApp
> ng serve "--port" "46403"

** Angular Live Development Server is listening on localhost:46403, open your browser on http://localhost:46403/ **

2019-10-22 21:19:20.885 +02:00 [Information] ℹ ｢wds｣: Project is running at http://localhost:46403/

2019-10-22 21:19:20.886 +02:00 [Information] ℹ ｢wds｣: webpack output is served from /

2019-10-22 21:19:21.474 +02:00 [Information] ℹ ｢wds｣: 404s will fallback to //index.html
ℹ ｢wdm｣: wait until bundle finished: /

2019-10-22 21:19:24.562 +02:00 [Information] Date: 2019-10-22T19:19:24.527Z
Hash: c632dcd6acac75f57081
Time: 7495ms
chunk {main} main.js, main.js.map (main) 514 kB [entry] [rendered]
chunk {polyfills} polyfills.js, polyfills.js.map (polyfills) 594 kB [entry] [rendered]
chunk {runtime} runtime.js, runtime.js.map (runtime) 5.22 kB [entry] [rendered]
chunk {scripts} scripts.js, scripts.js.map (scripts) 203 kB  [rendered]
chunk {styles} styles.css, styles.js, styles.css.map, styles.js.map (styles) 204 kB [entry] [rendered]
chunk {vendor} vendor.js, vendor.js.map (vendor) 4.92 MB [initial] [rendered]
ℹ ｢wdm｣: Compiled successfully.

2019-10-22 21:19:43.687 +02:00 [Information] Initialize RepositoryInspectorJobScheduler Run().
2019-10-22 21:22:20.803 +02:00 [Debug] Connection id ""0HLQN4KKEHMEV"" started.
2019-10-22 21:22:20.921 +02:00 [Debug] Connection id ""0HLQN4KKEHMEV"" received FIN.
2019-10-22 21:22:20.926 +02:00 [Debug] Failed to authenticate HTTPS connection.
System.Security.Authentication.AuthenticationException: Authentication failed, see inner exception. ---> Interop+OpenSsl+SslException: SSL Handshake failed with OpenSSL error - SSL_ERROR_SSL. ---> Interop+Crypto+OpenSslCryptographicException: error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown
   --- End of inner exception stack trace ---
   at Interop.OpenSsl.DoSslHandshake(SafeSslHandle context, Byte[] recvBuf, Int32 recvOffset, Int32 recvCount, Byte[]& sendBuf, Int32& sendCount)
   at System.Net.Security.SslStreamPal.HandshakeInternal(SafeFreeCredentials credential, SafeDeleteContext& context, SecurityBuffer inputBuffer, SecurityBuffer outputBuffer, SslAuthenticationOptions sslAuthenticationOptions)
   --- End of inner exception stack trace ---
   at System.Net.Security.SslState.StartSendAuthResetSignal(ProtocolToken message, AsyncProtocolRequest asyncRequest, ExceptionDispatchInfo exception)
   at System.Net.Security.SslState.CheckCompletionBeforeNextReceive(ProtocolToken message, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.StartSendBlob(Byte[] incoming, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.ProcessReceivedBlob(Byte[] buffer, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.StartReadFrame(Byte[] buffer, Int32 readBytes, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.PartialFrameCallback(AsyncProtocolRequest asyncRequest)
--- End of stack trace from previous location where exception was thrown ---
   at System.Net.Security.SslState.ThrowIfExceptional()
   at System.Net.Security.SslState.InternalEndProcessAuthentication(LazyAsyncResult lazyResult)
   at System.Net.Security.SslState.EndProcessAuthentication(IAsyncResult result)
   at System.Net.Security.SslStream.EndAuthenticateAsServer(IAsyncResult asyncResult)
   at System.Net.Security.SslStream.<>c.<AuthenticateAsServerAsync>b__51_1(IAsyncResult iar)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter.InnerOnConnectionAsync(ConnectionAdapterContext context)
2019-10-22 21:22:20.963 +02:00 [Debug] Connection id ""0HLQN4KKEHMEV"" sending FIN because: ""The client closed the connection.""
2019-10-22 21:22:20.964 +02:00 [Debug] Connection id ""0HLQN4KKEHMEV"" disconnecting.
2019-10-22 21:22:20.974 +02:00 [Debug] Connection id ""0HLQN4KKEHMEV"" stopped.
2019-10-22 21:22:21.101 +02:00 [Debug] Connection id ""0HLQN4KKEHMF0"" started.
2019-10-22 21:22:21.110 +02:00 [Debug] Connection id ""0HLQN4KKEHMF0"" received FIN.
2019-10-22 21:22:21.111 +02:00 [Debug] Failed to authenticate HTTPS connection.
System.Security.Authentication.AuthenticationException: Authentication failed, see inner exception. ---> Interop+OpenSsl+SslException: SSL Handshake failed with OpenSSL error - SSL_ERROR_SSL. ---> Interop+Crypto+OpenSslCryptographicException: error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown
   --- End of inner exception stack trace ---
   at Interop.OpenSsl.DoSslHandshake(SafeSslHandle context, Byte[] recvBuf, Int32 recvOffset, Int32 recvCount, Byte[]& sendBuf, Int32& sendCount)
   at System.Net.Security.SslStreamPal.HandshakeInternal(SafeFreeCredentials credential, SafeDeleteContext& context, SecurityBuffer inputBuffer, SecurityBuffer outputBuffer, SslAuthenticationOptions sslAuthenticationOptions)
   --- End of inner exception stack trace ---
   at System.Net.Security.SslState.StartSendAuthResetSignal(ProtocolToken message, AsyncProtocolRequest asyncRequest, ExceptionDispatchInfo exception)
   at System.Net.Security.SslState.CheckCompletionBeforeNextReceive(ProtocolToken message, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.StartSendBlob(Byte[] incoming, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.ProcessReceivedBlob(Byte[] buffer, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.StartReadFrame(Byte[] buffer, Int32 readBytes, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.PartialFrameCallback(AsyncProtocolRequest asyncRequest)
--- End of stack trace from previous location where exception was thrown ---
   at System.Net.Security.SslState.ThrowIfExceptional()
   at System.Net.Security.SslState.InternalEndProcessAuthentication(LazyAsyncResult lazyResult)
   at System.Net.Security.SslState.EndProcessAuthentication(IAsyncResult result)
   at System.Net.Security.SslStream.EndAuthenticateAsServer(IAsyncResult asyncResult)
   at System.Net.Security.SslStream.<>c.<AuthenticateAsServerAsync>b__51_1(IAsyncResult iar)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter.InnerOnConnectionAsync(ConnectionAdapterContext context)
2019-10-22 21:22:21.113 +02:00 [Debug] Connection id ""0HLQN4KKEHMF0"" sending FIN because: ""The client closed the connection.""
2019-10-22 21:22:21.114 +02:00 [Debug] Connection id ""0HLQN4KKEHMF0"" disconnecting.
2019-10-22 21:22:21.114 +02:00 [Debug] Connection id ""0HLQN4KKEHMF0"" stopped.
2019-10-22 21:22:23.004 +02:00 [Debug] Connection id ""0HLQN4KKEHMF1"" started.
2019-10-22 21:22:23.012 +02:00 [Debug] Connection id ""0HLQN4KKEHMF1"" received FIN.
2019-10-22 21:22:23.013 +02:00 [Debug] Failed to authenticate HTTPS connection.
System.Security.Authentication.AuthenticationException: Authentication failed, see inner exception. ---> Interop+OpenSsl+SslException: SSL Handshake failed with OpenSSL error - SSL_ERROR_SSL. ---> Interop+Crypto+OpenSslCryptographicException: error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown
   --- End of inner exception stack trace ---
   at Interop.OpenSsl.DoSslHandshake(SafeSslHandle context, Byte[] recvBuf, Int32 recvOffset, Int32 recvCount, Byte[]& sendBuf, Int32& sendCount)
   at System.Net.Security.SslStreamPal.HandshakeInternal(SafeFreeCredentials credential, SafeDeleteContext& context, SecurityBuffer inputBuffer, SecurityBuffer outputBuffer, SslAuthenticationOptions sslAuthenticationOptions)
   --- End of inner exception stack trace ---
   at System.Net.Security.SslState.StartSendAuthResetSignal(ProtocolToken message, AsyncProtocolRequest asyncRequest, ExceptionDispatchInfo exception)
   at System.Net.Security.SslState.CheckCompletionBeforeNextReceive(ProtocolToken message, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.StartSendBlob(Byte[] incoming, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.ProcessReceivedBlob(Byte[] buffer, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.StartReadFrame(Byte[] buffer, Int32 readBytes, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.PartialFrameCallback(AsyncProtocolRequest asyncRequest)
--- End of stack trace from previous location where exception was thrown ---
   at System.Net.Security.SslState.ThrowIfExceptional()
   at System.Net.Security.SslState.InternalEndProcessAuthentication(LazyAsyncResult lazyResult)
   at System.Net.Security.SslState.EndProcessAuthentication(IAsyncResult result)
   at System.Net.Security.SslStream.EndAuthenticateAsServer(IAsyncResult asyncResult)
   at System.Net.Security.SslStream.<>c.<AuthenticateAsServerAsync>b__51_1(IAsyncResult iar)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter.InnerOnConnectionAsync(ConnectionAdapterContext context)
2019-10-22 21:22:23.013 +02:00 [Debug] Connection id ""0HLQN4KKEHMF2"" started.
2019-10-22 21:22:23.014 +02:00 [Debug] Connection id ""0HLQN4KKEHMF1"" disconnecting.
2019-10-22 21:22:23.015 +02:00 [Debug] Connection id ""0HLQN4KKEHMF1"" sending FIN because: ""The client closed the connection.""
2019-10-22 21:22:23.015 +02:00 [Debug] Connection id ""0HLQN4KKEHMF1"" stopped.
2019-10-22 21:22:23.055 +02:00 [Information] Request starting HTTP/1.1 GET https://localhost:5001/  
2019-10-22 21:22:23.056 +02:00 [Debug] Wildcard detected, all requests with hosts will be allowed.
2019-10-22 21:22:23.082 +02:00 [Debug] AuthenticationScheme: "Cookies" was successfully authenticated.
2019-10-22 21:22:23.095 +02:00 [Debug] The request path "/" does not match a supported file type
2019-10-22 21:22:23.230 +02:00 [Debug] Request did not match any routes
2019-10-22 21:22:23.260 +02:00 [Debug] Connection id ""0HLQN4KKEHMF2"" completed keep alive response.
2019-10-22 21:22:23.263 +02:00 [Information] Request finished in 208.4971ms 200 text/html; charset=UTF-8
2019-10-22 21:22:23.299 +02:00 [Debug] Connection id ""0HLQN4KKEHMF3"" started.
2019-10-22 21:22:23.300 +02:00 [Debug] Connection id ""0HLQN4KKEHMF4"" started.
2019-10-22 21:22:23.300 +02:00 [Debug] Connection id ""0HLQN4KKEHMF5"" started.
2019-10-22 21:22:23.301 +02:00 [Debug] Connection id ""0HLQN4KKEHMF6"" started.
2019-10-22 21:22:23.301 +02:00 [Debug] Connection id ""0HLQN4KKEHMF7"" started.
2019-10-22 21:22:23.308 +02:00 [Debug] Failed to authenticate HTTPS connection.
System.Security.Authentication.AuthenticationException: Authentication failed, see inner exception. ---> Interop+OpenSsl+SslException: SSL Handshake failed with OpenSSL error - SSL_ERROR_SSL. ---> Interop+Crypto+OpenSslCryptographicException: error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown
   --- End of inner exception stack trace ---
   at Interop.OpenSsl.DoSslHandshake(SafeSslHandle context, Byte[] recvBuf, Int32 recvOffset, Int32 recvCount, Byte[]& sendBuf, Int32& sendCount)
   at System.Net.Security.SslStreamPal.HandshakeInternal(SafeFreeCredentials credential, SafeDeleteContext& context, SecurityBuffer inputBuffer, SecurityBuffer outputBuffer, SslAuthenticationOptions sslAuthenticationOptions)
   --- End of inner exception stack trace ---
   at System.Net.Security.SslState.StartSendAuthResetSignal(ProtocolToken message, AsyncProtocolRequest asyncRequest, ExceptionDispatchInfo exception)
   at System.Net.Security.SslState.CheckCompletionBeforeNextReceive(ProtocolToken message, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.StartSendBlob(Byte[] incoming, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.ProcessReceivedBlob(Byte[] buffer, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.StartReadFrame(Byte[] buffer, Int32 readBytes, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.PartialFrameCallback(AsyncProtocolRequest asyncRequest)
--- End of stack trace from previous location where exception was thrown ---
   at System.Net.Security.SslState.ThrowIfExceptional()
   at System.Net.Security.SslState.InternalEndProcessAuthentication(LazyAsyncResult lazyResult)
   at System.Net.Security.SslState.EndProcessAuthentication(IAsyncResult result)
   at System.Net.Security.SslStream.EndAuthenticateAsServer(IAsyncResult asyncResult)
   at System.Net.Security.SslStream.<>c.<AuthenticateAsServerAsync>b__51_1(IAsyncResult iar)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter.InnerOnConnectionAsync(ConnectionAdapterContext context)
2019-10-22 21:22:23.308 +02:00 [Debug] Failed to authenticate HTTPS connection.
System.Security.Authentication.AuthenticationException: Authentication failed, see inner exception. ---> Interop+OpenSsl+SslException: SSL Handshake failed with OpenSSL error - SSL_ERROR_SSL. ---> Interop+Crypto+OpenSslCryptographicException: error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown
   --- End of inner exception stack trace ---
   at Interop.OpenSsl.DoSslHandshake(SafeSslHandle context, Byte[] recvBuf, Int32 recvOffset, Int32 recvCount, Byte[]& sendBuf, Int32& sendCount)
   at System.Net.Security.SslStreamPal.HandshakeInternal(SafeFreeCredentials credential, SafeDeleteContext& context, SecurityBuffer inputBuffer, SecurityBuffer outputBuffer, SslAuthenticationOptions sslAuthenticationOptions)
   --- End of inner exception stack trace ---
   at System.Net.Security.SslState.StartSendAuthResetSignal(ProtocolToken message, AsyncProtocolRequest asyncRequest, ExceptionDispatchInfo exception)
   at System.Net.Security.SslState.CheckCompletionBeforeNextReceive(ProtocolToken message, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.StartSendBlob(Byte[] incoming, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.ProcessReceivedBlob(Byte[] buffer, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.StartReadFrame(Byte[] buffer, Int32 readBytes, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.PartialFrameCallback(AsyncProtocolRequest asyncRequest)
--- End of stack trace from previous location where exception was thrown ---
   at System.Net.Security.SslState.ThrowIfExceptional()
   at System.Net.Security.SslState.InternalEndProcessAuthentication(LazyAsyncResult lazyResult)
   at System.Net.Security.SslState.EndProcessAuthentication(IAsyncResult result)
   at System.Net.Security.SslStream.EndAuthenticateAsServer(IAsyncResult asyncResult)
   at System.Net.Security.SslStream.<>c.<AuthenticateAsServerAsync>b__51_1(IAsyncResult iar)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter.InnerOnConnectionAsync(ConnectionAdapterContext context)
2019-10-22 21:22:23.309 +02:00 [Debug] Connection id ""0HLQN4KKEHMF4"" received FIN.
2019-10-22 21:22:23.310 +02:00 [Debug] Connection id ""0HLQN4KKEHMF3"" disconnecting.
2019-10-22 21:22:23.311 +02:00 [Information] Request starting HTTP/1.1 GET https://localhost:5001/styles.css  
2019-10-22 21:22:23.312 +02:00 [Debug] Connection id ""0HLQN4KKEHMF3"" sending FIN because: ""The Socket transport's send loop completed gracefully.""
2019-10-22 21:22:23.312 +02:00 [Debug] Failed to authenticate HTTPS connection.
System.Security.Authentication.AuthenticationException: Authentication failed, see inner exception. ---> Interop+OpenSsl+SslException: SSL Handshake failed with OpenSSL error - SSL_ERROR_SSL. ---> Interop+Crypto+OpenSslCryptographicException: error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown
   --- End of inner exception stack trace ---
   at Interop.OpenSsl.DoSslHandshake(SafeSslHandle context, Byte[] recvBuf, Int32 recvOffset, Int32 recvCount, Byte[]& sendBuf, Int32& sendCount)
   at System.Net.Security.SslStreamPal.HandshakeInternal(SafeFreeCredentials credential, SafeDeleteContext& context, SecurityBuffer inputBuffer, SecurityBuffer outputBuffer, SslAuthenticationOptions sslAuthenticationOptions)
   --- End of inner exception stack trace ---
   at System.Net.Security.SslState.StartSendAuthResetSignal(ProtocolToken message, AsyncProtocolRequest asyncRequest, ExceptionDispatchInfo exception)
   at System.Net.Security.SslState.CheckCompletionBeforeNextReceive(ProtocolToken message, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.StartSendBlob(Byte[] incoming, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.ProcessReceivedBlob(Byte[] buffer, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.StartReadFrame(Byte[] buffer, Int32 readBytes, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.PartialFrameCallback(AsyncProtocolRequest asyncRequest)
--- End of stack trace from previous location where exception was thrown ---
   at System.Net.Security.SslState.ThrowIfExceptional()
   at System.Net.Security.SslState.InternalEndProcessAuthentication(LazyAsyncResult lazyResult)
   at System.Net.Security.SslState.EndProcessAuthentication(IAsyncResult result)
   at System.Net.Security.SslStream.EndAuthenticateAsServer(IAsyncResult asyncResult)
   at System.Net.Security.SslStream.<>c.<AuthenticateAsServerAsync>b__51_1(IAsyncResult iar)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter.InnerOnConnectionAsync(ConnectionAdapterContext context)
2019-10-22 21:22:23.309 +02:00 [Debug] Failed to authenticate HTTPS connection.
System.Security.Authentication.AuthenticationException: Authentication failed, see inner exception. ---> Interop+OpenSsl+SslException: SSL Handshake failed with OpenSSL error - SSL_ERROR_SSL. ---> Interop+Crypto+OpenSslCryptographicException: error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown
   --- End of inner exception stack trace ---
   at Interop.OpenSsl.DoSslHandshake(SafeSslHandle context, Byte[] recvBuf, Int32 recvOffset, Int32 recvCount, Byte[]& sendBuf, Int32& sendCount)
   at System.Net.Security.SslStreamPal.HandshakeInternal(SafeFreeCredentials credential, SafeDeleteContext& context, SecurityBuffer inputBuffer, SecurityBuffer outputBuffer, SslAuthenticationOptions sslAuthenticationOptions)
   --- End of inner exception stack trace ---
   at System.Net.Security.SslState.StartSendAuthResetSignal(ProtocolToken message, AsyncProtocolRequest asyncRequest, ExceptionDispatchInfo exception)
   at System.Net.Security.SslState.CheckCompletionBeforeNextReceive(ProtocolToken message, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.StartSendBlob(Byte[] incoming, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.ProcessReceivedBlob(Byte[] buffer, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.StartReadFrame(Byte[] buffer, Int32 readBytes, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.PartialFrameCallback(AsyncProtocolRequest asyncRequest)
--- End of stack trace from previous location where exception was thrown ---
   at System.Net.Security.SslState.ThrowIfExceptional()
   at System.Net.Security.SslState.InternalEndProcessAuthentication(LazyAsyncResult lazyResult)
   at System.Net.Security.SslState.EndProcessAuthentication(IAsyncResult result)
   at System.Net.Security.SslStream.EndAuthenticateAsServer(IAsyncResult asyncResult)
   at System.Net.Security.SslStream.<>c.<AuthenticateAsServerAsync>b__51_1(IAsyncResult iar)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter.InnerOnConnectionAsync(ConnectionAdapterContext context)
2019-10-22 21:22:23.309 +02:00 [Debug] Connection id ""0HLQN4KKEHMF8"" started.
2019-10-22 21:22:23.309 +02:00 [Debug] Connection id ""0HLQN4KKEHMF5"" received FIN.
2019-10-22 21:22:23.311 +02:00 [Debug] Connection id ""0HLQN4KKEHMF5"" disconnecting.
2019-10-22 21:22:23.311 +02:00 [Debug] Connection id ""0HLQN4KKEHMF6"" received FIN.
2019-10-22 21:22:23.309 +02:00 [Debug] Connection id ""0HLQN4KKEHMF3"" received FIN.
2019-10-22 21:22:23.313 +02:00 [Debug] AuthenticationScheme: "Cookies" was successfully authenticated.
2019-10-22 21:22:23.317 +02:00 [Debug] Connection id ""0HLQN4KKEHMF7"" received FIN.
2019-10-22 21:22:23.317 +02:00 [Debug] Failed to authenticate HTTPS connection.
System.Security.Authentication.AuthenticationException: Authentication failed, see inner exception. ---> Interop+OpenSsl+SslException: SSL Handshake failed with OpenSSL error - SSL_ERROR_SSL. ---> Interop+Crypto+OpenSslCryptographicException: error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown
   --- End of inner exception stack trace ---
   at Interop.OpenSsl.DoSslHandshake(SafeSslHandle context, Byte[] recvBuf, Int32 recvOffset, Int32 recvCount, Byte[]& sendBuf, Int32& sendCount)
   at System.Net.Security.SslStreamPal.HandshakeInternal(SafeFreeCredentials credential, SafeDeleteContext& context, SecurityBuffer inputBuffer, SecurityBuffer outputBuffer, SslAuthenticationOptions sslAuthenticationOptions)
   --- End of inner exception stack trace ---
   at System.Net.Security.SslState.StartSendAuthResetSignal(ProtocolToken message, AsyncProtocolRequest asyncRequest, ExceptionDispatchInfo exception)
   at System.Net.Security.SslState.CheckCompletionBeforeNextReceive(ProtocolToken message, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.StartSendBlob(Byte[] incoming, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.ProcessReceivedBlob(Byte[] buffer, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.StartReadFrame(Byte[] buffer, Int32 readBytes, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.PartialFrameCallback(AsyncProtocolRequest asyncRequest)
--- End of stack trace from previous location where exception was thrown ---
   at System.Net.Security.SslState.ThrowIfExceptional()
   at System.Net.Security.SslState.InternalEndProcessAuthentication(LazyAsyncResult lazyResult)
   at System.Net.Security.SslState.EndProcessAuthentication(IAsyncResult result)
   at System.Net.Security.SslStream.EndAuthenticateAsServer(IAsyncResult asyncResult)
   at System.Net.Security.SslStream.<>c.<AuthenticateAsServerAsync>b__51_1(IAsyncResult iar)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter.InnerOnConnectionAsync(ConnectionAdapterContext context)
2019-10-22 21:22:23.319 +02:00 [Debug] Connection id ""0HLQN4KKEHMF6"" disconnecting.
2019-10-22 21:22:23.320 +02:00 [Debug] Connection id ""0HLQN4KKEHMF3"" stopped.
2019-10-22 21:22:23.321 +02:00 [Debug] Connection id ""0HLQN4KKEHMF4"" disconnecting.
2019-10-22 21:22:23.321 +02:00 [Debug] Connection id ""0HLQN4KKEHMF9"" started.
2019-10-22 21:22:23.322 +02:00 [Debug] Connection id ""0HLQN4KKEHMF7"" disconnecting.
2019-10-22 21:22:23.322 +02:00 [Debug] Connection id ""0HLQN4KKEHMF7"" sending FIN because: ""The Socket transport's send loop completed gracefully.""
2019-10-22 21:22:23.325 +02:00 [Debug] Connection id ""0HLQN4KKEHMF7"" stopped.
2019-10-22 21:22:23.327 +02:00 [Debug] The request path "/styles.css" does not match an existing file
2019-10-22 21:22:23.328 +02:00 [Debug] Connection id ""0HLQN4KKEHMF5"" sending FIN because: ""The client closed the connection.""
2019-10-22 21:22:23.329 +02:00 [Debug] Connection id ""0HLQN4KKEHMF6"" sending FIN because: ""The client closed the connection.""
2019-10-22 21:22:23.329 +02:00 [Debug] Connection id ""0HLQN4KKEHMF4"" sending FIN because: ""The Socket transport's send loop completed gracefully.""
2019-10-22 21:22:23.329 +02:00 [Debug] Connection id ""0HLQN4KKEHMF8"" received FIN.
2019-10-22 21:22:23.329 +02:00 [Debug] Connection id ""0HLQN4KKEHMFA"" started.
2019-10-22 21:22:23.330 +02:00 [Debug] Connection id ""0HLQN4KKEHMF5"" stopped.
2019-10-22 21:22:23.330 +02:00 [Debug] Failed to authenticate HTTPS connection.
System.Security.Authentication.AuthenticationException: Authentication failed, see inner exception. ---> Interop+OpenSsl+SslException: SSL Handshake failed with OpenSSL error - SSL_ERROR_SSL. ---> Interop+Crypto+OpenSslCryptographicException: error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown
   --- End of inner exception stack trace ---
   at Interop.OpenSsl.DoSslHandshake(SafeSslHandle context, Byte[] recvBuf, Int32 recvOffset, Int32 recvCount, Byte[]& sendBuf, Int32& sendCount)
   at System.Net.Security.SslStreamPal.HandshakeInternal(SafeFreeCredentials credential, SafeDeleteContext& context, SecurityBuffer inputBuffer, SecurityBuffer outputBuffer, SslAuthenticationOptions sslAuthenticationOptions)
   --- End of inner exception stack trace ---
   at System.Net.Security.SslState.StartSendAuthResetSignal(ProtocolToken message, AsyncProtocolRequest asyncRequest, ExceptionDispatchInfo exception)
   at System.Net.Security.SslState.CheckCompletionBeforeNextReceive(ProtocolToken message, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.StartSendBlob(Byte[] incoming, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.ProcessReceivedBlob(Byte[] buffer, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.StartReadFrame(Byte[] buffer, Int32 readBytes, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.PartialFrameCallback(AsyncProtocolRequest asyncRequest)
--- End of stack trace from previous location where exception was thrown ---
   at System.Net.Security.SslState.ThrowIfExceptional()
   at System.Net.Security.SslState.InternalEndProcessAuthentication(LazyAsyncResult lazyResult)
   at System.Net.Security.SslState.EndProcessAuthentication(IAsyncResult result)
   at System.Net.Security.SslStream.EndAuthenticateAsServer(IAsyncResult asyncResult)
   at System.Net.Security.SslStream.<>c.<AuthenticateAsServerAsync>b__51_1(IAsyncResult iar)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter.InnerOnConnectionAsync(ConnectionAdapterContext context)
2019-10-22 21:22:23.330 +02:00 [Debug] Connection id ""0HLQN4KKEHMF6"" stopped.
2019-10-22 21:22:23.330 +02:00 [Debug] Connection id ""0HLQN4KKEHMF4"" stopped.
2019-10-22 21:22:23.330 +02:00 [Debug] Connection id ""0HLQN4KKEHMFB"" started.
2019-10-22 21:22:23.331 +02:00 [Debug] Request successfully matched the route with name '"default"' and template '"{controller}/{action=Index}/{id?}"'
2019-10-22 21:22:23.336 +02:00 [Debug] Connection id ""0HLQN4KKEHMF9"" received FIN.
2019-10-22 21:22:23.337 +02:00 [Debug] Failed to authenticate HTTPS connection.
System.Security.Authentication.AuthenticationException: Authentication failed, see inner exception. ---> Interop+OpenSsl+SslException: SSL Handshake failed with OpenSSL error - SSL_ERROR_SSL. ---> Interop+Crypto+OpenSslCryptographicException: error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown
   --- End of inner exception stack trace ---
   at Interop.OpenSsl.DoSslHandshake(SafeSslHandle context, Byte[] recvBuf, Int32 recvOffset, Int32 recvCount, Byte[]& sendBuf, Int32& sendCount)
   at System.Net.Security.SslStreamPal.HandshakeInternal(SafeFreeCredentials credential, SafeDeleteContext& context, SecurityBuffer inputBuffer, SecurityBuffer outputBuffer, SslAuthenticationOptions sslAuthenticationOptions)
   --- End of inner exception stack trace ---
   at System.Net.Security.SslState.StartSendAuthResetSignal(ProtocolToken message, AsyncProtocolRequest asyncRequest, ExceptionDispatchInfo exception)
   at System.Net.Security.SslState.CheckCompletionBeforeNextReceive(ProtocolToken message, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.StartSendBlob(Byte[] incoming, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.ProcessReceivedBlob(Byte[] buffer, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.StartReadFrame(Byte[] buffer, Int32 readBytes, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.PartialFrameCallback(AsyncProtocolRequest asyncRequest)
--- End of stack trace from previous location where exception was thrown ---
   at System.Net.Security.SslState.ThrowIfExceptional()
   at System.Net.Security.SslState.InternalEndProcessAuthentication(LazyAsyncResult lazyResult)
   at System.Net.Security.SslState.EndProcessAuthentication(IAsyncResult result)
   at System.Net.Security.SslStream.EndAuthenticateAsServer(IAsyncResult asyncResult)
   at System.Net.Security.SslStream.<>c.<AuthenticateAsServerAsync>b__51_1(IAsyncResult iar)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter.InnerOnConnectionAsync(ConnectionAdapterContext context)
2019-10-22 21:22:23.340 +02:00 [Information] Request starting HTTP/1.1 GET https://localhost:5001/polyfills.js  
2019-10-22 21:22:23.342 +02:00 [Debug] Connection id ""0HLQN4KKEHMF8"" disconnecting.
2019-10-22 21:22:23.343 +02:00 [Debug] Connection id ""0HLQN4KKEHMFC"" started.
2019-10-22 21:22:23.344 +02:00 [Debug] Connection id ""0HLQN4KKEHMF9"" disconnecting.
2019-10-22 21:22:23.344 +02:00 [Debug] Connection id ""0HLQN4KKEHMF8"" sending FIN because: ""The Socket transport's send loop completed gracefully.""
2019-10-22 21:22:23.344 +02:00 [Debug] Connection id ""0HLQN4KKEHMFD"" started.
2019-10-22 21:22:23.345 +02:00 [Debug] AuthenticationScheme: "Cookies" was successfully authenticated.
2019-10-22 21:22:23.345 +02:00 [Debug] Connection id ""0HLQN4KKEHMF9"" sending FIN because: ""The Socket transport's send loop completed gracefully.""
2019-10-22 21:22:23.345 +02:00 [Debug] Connection id ""0HLQN4KKEHMF8"" stopped.
2019-10-22 21:22:23.345 +02:00 [Debug] The request path "/polyfills.js" does not match an existing file
2019-10-22 21:22:23.346 +02:00 [Debug] Connection id ""0HLQN4KKEHMFE"" started.
2019-10-22 21:22:23.346 +02:00 [Debug] No actions matched the current request. Route values: ["controller=styles.css", "action=Index"]
2019-10-22 21:22:23.348 +02:00 [Debug] Connection id ""0HLQN4KKEHMF9"" stopped.
2019-10-22 21:22:23.349 +02:00 [Debug] Request successfully matched the route with name '"default"' and template '"{controller}/{action=Index}/{id?}"'
2019-10-22 21:22:23.351 +02:00 [Debug] Connection id ""0HLQN4KKEHMFB"" received FIN.
2019-10-22 21:22:23.352 +02:00 [Debug] Connection id ""0HLQN4KKEHMFC"" received FIN.
2019-10-22 21:22:23.352 +02:00 [Debug] Failed to authenticate HTTPS connection.
System.Security.Authentication.AuthenticationException: Authentication failed, see inner exception. ---> Interop+OpenSsl+SslException: SSL Handshake failed with OpenSSL error - SSL_ERROR_SSL. ---> Interop+Crypto+OpenSslCryptographicException: error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown
   --- End of inner exception stack trace ---
   at Interop.OpenSsl.DoSslHandshake(SafeSslHandle context, Byte[] recvBuf, Int32 recvOffset, Int32 recvCount, Byte[]& sendBuf, Int32& sendCount)
   at System.Net.Security.SslStreamPal.HandshakeInternal(SafeFreeCredentials credential, SafeDeleteContext& context, SecurityBuffer inputBuffer, SecurityBuffer outputBuffer, SslAuthenticationOptions sslAuthenticationOptions)
   --- End of inner exception stack trace ---
   at System.Net.Security.SslState.StartSendAuthResetSignal(ProtocolToken message, AsyncProtocolRequest asyncRequest, ExceptionDispatchInfo exception)
   at System.Net.Security.SslState.CheckCompletionBeforeNextReceive(ProtocolToken message, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.StartSendBlob(Byte[] incoming, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.ProcessReceivedBlob(Byte[] buffer, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.StartReadFrame(Byte[] buffer, Int32 readBytes, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.PartialFrameCallback(AsyncProtocolRequest asyncRequest)
--- End of stack trace from previous location where exception was thrown ---
   at System.Net.Security.SslState.ThrowIfExceptional()
   at System.Net.Security.SslState.InternalEndProcessAuthentication(LazyAsyncResult lazyResult)
   at System.Net.Security.SslState.EndProcessAuthentication(IAsyncResult result)
   at System.Net.Security.SslStream.EndAuthenticateAsServer(IAsyncResult asyncResult)
   at System.Net.Security.SslStream.<>c.<AuthenticateAsServerAsync>b__51_1(IAsyncResult iar)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter.InnerOnConnectionAsync(ConnectionAdapterContext context)
2019-10-22 21:22:23.353 +02:00 [Debug] Connection id ""0HLQN4KKEHMFF"" started.
2019-10-22 21:22:23.353 +02:00 [Debug] No actions matched the current request. Route values: ["controller=styles.css", "action=Index"]
2019-10-22 21:22:23.352 +02:00 [Debug] Failed to authenticate HTTPS connection.
System.Security.Authentication.AuthenticationException: Authentication failed, see inner exception. ---> Interop+OpenSsl+SslException: SSL Handshake failed with OpenSSL error - SSL_ERROR_SSL. ---> Interop+Crypto+OpenSslCryptographicException: error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown
   --- End of inner exception stack trace ---
   at Interop.OpenSsl.DoSslHandshake(SafeSslHandle context, Byte[] recvBuf, Int32 recvOffset, Int32 recvCount, Byte[]& sendBuf, Int32& sendCount)
   at System.Net.Security.SslStreamPal.HandshakeInternal(SafeFreeCredentials credential, SafeDeleteContext& context, SecurityBuffer inputBuffer, SecurityBuffer outputBuffer, SslAuthenticationOptions sslAuthenticationOptions)
   --- End of inner exception stack trace ---
   at System.Net.Security.SslState.StartSendAuthResetSignal(ProtocolToken message, AsyncProtocolRequest asyncRequest, ExceptionDispatchInfo exception)
   at System.Net.Security.SslState.CheckCompletionBeforeNextReceive(ProtocolToken message, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.StartSendBlob(Byte[] incoming, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.ProcessReceivedBlob(Byte[] buffer, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.StartReadFrame(Byte[] buffer, Int32 readBytes, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.PartialFrameCallback(AsyncProtocolRequest asyncRequest)
--- End of stack trace from previous location where exception was thrown ---
   at System.Net.Security.SslState.ThrowIfExceptional()
   at System.Net.Security.SslState.InternalEndProcessAuthentication(LazyAsyncResult lazyResult)
   at System.Net.Security.SslState.EndProcessAuthentication(IAsyncResult result)
   at System.Net.Security.SslStream.EndAuthenticateAsServer(IAsyncResult asyncResult)
   at System.Net.Security.SslStream.<>c.<AuthenticateAsServerAsync>b__51_1(IAsyncResult iar)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter.InnerOnConnectionAsync(ConnectionAdapterContext context)
2019-10-22 21:22:23.354 +02:00 [Debug] Connection id ""0HLQN4KKEHMFC"" disconnecting.
2019-10-22 21:22:23.355 +02:00 [Debug] Request did not match any routes
2019-10-22 21:22:23.355 +02:00 [Information] Request starting HTTP/1.1 GET https://localhost:5001/vendor.js  
2019-10-22 21:22:23.356 +02:00 [Debug] No actions matched the current request. Route values: ["controller=polyfills.js", "action=Index"]
2019-10-22 21:22:23.356 +02:00 [Debug] Connection id ""0HLQN4KKEHMFG"" started.
2019-10-22 21:22:23.358 +02:00 [Debug] Connection id ""0HLQN4KKEHMFB"" disconnecting.
2019-10-22 21:22:23.358 +02:00 [Debug] Connection id ""0HLQN4KKEHMFC"" sending FIN because: ""The Socket transport's send loop completed gracefully.""
2019-10-22 21:22:23.358 +02:00 [Debug] No actions matched the current request. Route values: ["controller=polyfills.js", "action=Index"]
2019-10-22 21:22:23.358 +02:00 [Debug] AuthenticationScheme: "Cookies" was successfully authenticated.
2019-10-22 21:22:23.359 +02:00 [Debug] Connection id ""0HLQN4KKEHMFB"" sending FIN because: ""The Socket transport's send loop completed gracefully.""
2019-10-22 21:22:23.359 +02:00 [Debug] Connection id ""0HLQN4KKEHMFC"" stopped.
2019-10-22 21:22:23.359 +02:00 [Debug] Connection id ""0HLQN4KKEHMFB"" stopped.
2019-10-22 21:22:23.359 +02:00 [Debug] Request did not match any routes
2019-10-22 21:22:23.360 +02:00 [Debug] The request path "/vendor.js" does not match an existing file
2019-10-22 21:22:23.361 +02:00 [Debug] Request successfully matched the route with name '"default"' and template '"{controller}/{action=Index}/{id?}"'
2019-10-22 21:22:23.361 +02:00 [Debug] No actions matched the current request. Route values: ["controller=vendor.js", "action=Index"]
2019-10-22 21:22:23.361 +02:00 [Debug] No actions matched the current request. Route values: ["controller=vendor.js", "action=Index"]
2019-10-22 21:22:23.362 +02:00 [Information] Request starting HTTP/1.1 GET https://localhost:5001/main.js  
2019-10-22 21:22:23.362 +02:00 [Debug] Request did not match any routes
2019-10-22 21:22:23.362 +02:00 [Debug] AuthenticationScheme: "Cookies" was successfully authenticated.
2019-10-22 21:22:23.363 +02:00 [Debug] The request path "/main.js" does not match an existing file
2019-10-22 21:22:23.363 +02:00 [Debug] Request successfully matched the route with name '"default"' and template '"{controller}/{action=Index}/{id?}"'
2019-10-22 21:22:23.363 +02:00 [Debug] No actions matched the current request. Route values: ["controller=main.js", "action=Index"]
2019-10-22 21:22:23.365 +02:00 [Information] Request starting HTTP/1.1 GET https://localhost:5001/runtime.js  
2019-10-22 21:22:23.366 +02:00 [Information] Request starting HTTP/1.1 GET https://localhost:5001/styles.js  
2019-10-22 21:22:23.372 +02:00 [Debug] Connection id ""0HLQN4KKEHMFA"" completed keep alive response.
2019-10-22 21:22:23.372 +02:00 [Debug] No actions matched the current request. Route values: ["controller=main.js", "action=Index"]
2019-10-22 21:22:23.373 +02:00 [Debug] AuthenticationScheme: "Cookies" was successfully authenticated.
2019-10-22 21:22:23.373 +02:00 [Debug] Connection id ""0HLQN4KKEHMF2"" completed keep alive response.
2019-10-22 21:22:23.374 +02:00 [Debug] AuthenticationScheme: "Cookies" was successfully authenticated.
2019-10-22 21:22:23.374 +02:00 [Information] Request finished in 34.2149ms 200 application/javascript; charset=UTF-8
2019-10-22 21:22:23.374 +02:00 [Debug] Request did not match any routes
2019-10-22 21:22:23.374 +02:00 [Information] Request finished in 63.0414ms 200 text/css; charset=UTF-8
2019-10-22 21:22:23.377 +02:00 [Information] Request starting HTTP/1.1 GET https://localhost:5001/assets/favicon.ico  
2019-10-22 21:22:23.377 +02:00 [Debug] The request path "/styles.js" does not match an existing file
2019-10-22 21:22:23.377 +02:00 [Debug] The request path "/runtime.js" does not match an existing file
2019-10-22 21:22:23.377 +02:00 [Information] Request starting HTTP/1.1 GET https://localhost:5001/scripts.js  
2019-10-22 21:22:23.378 +02:00 [Debug] AuthenticationScheme: "Cookies" was not authenticated.
2019-10-22 21:22:23.378 +02:00 [Debug] Request successfully matched the route with name '"default"' and template '"{controller}/{action=Index}/{id?}"'
2019-10-22 21:22:23.378 +02:00 [Debug] Request successfully matched the route with name '"default"' and template '"{controller}/{action=Index}/{id?}"'
2019-10-22 21:22:23.379 +02:00 [Debug] The request path "/assets/favicon.ico" does not match an existing file
2019-10-22 21:22:23.379 +02:00 [Debug] AuthenticationScheme: "Cookies" was successfully authenticated.
2019-10-22 21:22:23.379 +02:00 [Debug] No actions matched the current request. Route values: ["controller=runtime.js", "action=Index"]
2019-10-22 21:22:23.384 +02:00 [Debug] Connection id ""0HLQN4KKEHMFE"" completed keep alive response.
2019-10-22 21:22:23.386 +02:00 [Debug] Connection id ""0HLQN4KKEHMFD"" completed keep alive response.
2019-10-22 21:22:23.391 +02:00 [Debug] No actions matched the current request. Route values: ["controller=styles.js", "action=Index"]
2019-10-22 21:22:23.391 +02:00 [Debug] Request successfully matched the route with name '"default"' and template '"{controller}/{action=Index}/{id?}"'
2019-10-22 21:22:23.391 +02:00 [Debug] The request path "/scripts.js" does not match an existing file
2019-10-22 21:22:23.392 +02:00 [Debug] No actions matched the current request. Route values: ["controller=runtime.js", "action=Index"]
2019-10-22 21:22:23.392 +02:00 [Debug] Request successfully matched the route with name '"default"' and template '"{controller}/{action=Index}/{id?}"'
2019-10-22 21:22:23.392 +02:00 [Debug] Request did not match any routes
2019-10-22 21:22:23.392 +02:00 [Information] Request finished in 30.8573ms 200 application/javascript; charset=UTF-8
2019-10-22 21:22:23.405 +02:00 [Debug] No actions matched the current request. Route values: ["controller=styles.js", "action=Index"]
2019-10-22 21:22:23.406 +02:00 [Debug] No actions matched the current request. Route values: ["controller=assets", "action=favicon.ico"]
2019-10-22 21:22:23.406 +02:00 [Debug] No actions matched the current request. Route values: ["controller=scripts.js", "action=Index"]
2019-10-22 21:22:23.406 +02:00 [Debug] No actions matched the current request. Route values: ["controller=scripts.js", "action=Index"]
2019-10-22 21:22:23.407 +02:00 [Debug] Connection id ""0HLQN4KKEHMFF"" completed keep alive response.
2019-10-22 21:22:23.407 +02:00 [Debug] Request did not match any routes
2019-10-22 21:22:23.406 +02:00 [Information] Request finished in 50.8223ms 200 application/javascript; charset=UTF-8
2019-10-22 21:22:23.408 +02:00 [Information] Request finished in 42.2288ms 200 application/javascript; charset=UTF-8
2019-10-22 21:22:23.408 +02:00 [Debug] Request did not match any routes
2019-10-22 21:22:23.409 +02:00 [Debug] No actions matched the current request. Route values: ["controller=assets", "action=favicon.ico"]
2019-10-22 21:22:23.409 +02:00 [Debug] Request did not match any routes
2019-10-22 21:22:23.411 +02:00 [Debug] Connection id ""0HLQN4KKEHMFG"" completed keep alive response.
2019-10-22 21:22:23.411 +02:00 [Debug] Connection id ""0HLQN4KKEHMFA"" completed keep alive response.
2019-10-22 21:22:23.411 +02:00 [Debug] Connection id ""0HLQN4KKEHMF2"" completed keep alive response.
2019-10-22 21:22:23.412 +02:00 [Information] Request finished in 45.6354ms 200 application/javascript; charset=UTF-8
2019-10-22 21:22:23.412 +02:00 [Information] Request finished in 34.653ms 200 application/javascript; charset=UTF-8
2019-10-22 21:22:23.412 +02:00 [Information] Request finished in 35.8441ms 200 image/vnd.microsoft.icon; charset=UTF-8
2019-10-22 21:22:23.517 +02:00 [Information] Request starting HTTP/1.1 GET https://localhost:5001/sockjs-node/info?t=1571772143513  
2019-10-22 21:22:23.518 +02:00 [Debug] AuthenticationScheme: "Cookies" was successfully authenticated.
2019-10-22 21:22:23.519 +02:00 [Debug] The request path "/sockjs-node/info" does not match a supported file type
2019-10-22 21:22:23.519 +02:00 [Debug] Request successfully matched the route with name '"default"' and template '"{controller}/{action=Index}/{id?}"'
2019-10-22 21:22:23.520 +02:00 [Debug] No actions matched the current request. Route values: ["controller=sockjs-node", "action=info"]
2019-10-22 21:22:23.520 +02:00 [Debug] No actions matched the current request. Route values: ["controller=sockjs-node", "action=info"]
2019-10-22 21:22:23.521 +02:00 [Debug] Request did not match any routes
2019-10-22 21:22:23.530 +02:00 [Debug] Connection id ""0HLQN4KKEHMFD"" completed keep alive response.
2019-10-22 21:22:23.530 +02:00 [Information] Request finished in 12.4816ms 200 application/json; charset=UTF-8
2019-10-22 21:22:23.549 +02:00 [Debug] Connection id ""0HLQN4KKEHMFH"" started.
2019-10-22 21:22:23.550 +02:00 [Information] Request starting HTTP/1.1 GET https://localhost:5001/sockjs-node/info?t=1571772143547  
2019-10-22 21:22:23.551 +02:00 [Debug] AuthenticationScheme: "Cookies" was successfully authenticated.
2019-10-22 21:22:23.551 +02:00 [Debug] The request path "/sockjs-node/info" does not match a supported file type
2019-10-22 21:22:23.551 +02:00 [Debug] Request successfully matched the route with name '"default"' and template '"{controller}/{action=Index}/{id?}"'
2019-10-22 21:22:23.552 +02:00 [Debug] No actions matched the current request. Route values: ["controller=sockjs-node", "action=info"]
2019-10-22 21:22:23.552 +02:00 [Debug] No actions matched the current request. Route values: ["controller=sockjs-node", "action=info"]
2019-10-22 21:22:23.553 +02:00 [Debug] Request did not match any routes
2019-10-22 21:22:23.554 +02:00 [Debug] Connection id ""0HLQN4KKEHMFD"" completed keep alive response.
2019-10-22 21:22:23.555 +02:00 [Information] Request finished in 4.7651ms 200 application/json; charset=UTF-8
2019-10-22 21:22:23.561 +02:00 [Debug] Connection id ""0HLQN4KKEHMFH"" received FIN.
2019-10-22 21:22:23.561 +02:00 [Debug] Failed to authenticate HTTPS connection.
System.Security.Authentication.AuthenticationException: Authentication failed, see inner exception. ---> Interop+OpenSsl+SslException: SSL Handshake failed with OpenSSL error - SSL_ERROR_SSL. ---> Interop+Crypto+OpenSslCryptographicException: error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown
   --- End of inner exception stack trace ---
   at Interop.OpenSsl.DoSslHandshake(SafeSslHandle context, Byte[] recvBuf, Int32 recvOffset, Int32 recvCount, Byte[]& sendBuf, Int32& sendCount)
   at System.Net.Security.SslStreamPal.HandshakeInternal(SafeFreeCredentials credential, SafeDeleteContext& context, SecurityBuffer inputBuffer, SecurityBuffer outputBuffer, SslAuthenticationOptions sslAuthenticationOptions)
   --- End of inner exception stack trace ---
   at System.Net.Security.SslState.StartSendAuthResetSignal(ProtocolToken message, AsyncProtocolRequest asyncRequest, ExceptionDispatchInfo exception)
   at System.Net.Security.SslState.CheckCompletionBeforeNextReceive(ProtocolToken message, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.StartSendBlob(Byte[] incoming, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.ProcessReceivedBlob(Byte[] buffer, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.StartReadFrame(Byte[] buffer, Int32 readBytes, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.PartialFrameCallback(AsyncProtocolRequest asyncRequest)
--- End of stack trace from previous location where exception was thrown ---
   at System.Net.Security.SslState.ThrowIfExceptional()
   at System.Net.Security.SslState.InternalEndProcessAuthentication(LazyAsyncResult lazyResult)
   at System.Net.Security.SslState.EndProcessAuthentication(IAsyncResult result)
   at System.Net.Security.SslStream.EndAuthenticateAsServer(IAsyncResult asyncResult)
   at System.Net.Security.SslStream.<>c.<AuthenticateAsServerAsync>b__51_1(IAsyncResult iar)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter.InnerOnConnectionAsync(ConnectionAdapterContext context)
2019-10-22 21:22:23.563 +02:00 [Debug] Connection id ""0HLQN4KKEHMFH"" disconnecting.
2019-10-22 21:22:23.563 +02:00 [Debug] Connection id ""0HLQN4KKEHMFH"" sending FIN because: ""The client closed the connection.""
2019-10-22 21:22:23.564 +02:00 [Debug] Connection id ""0HLQN4KKEHMFH"" stopped.
2019-10-22 21:22:23.574 +02:00 [Debug] Connection id ""0HLQN4KKEHMFI"" started.
2019-10-22 21:22:23.581 +02:00 [Debug] Connection id ""0HLQN4KKEHMFI"" received FIN.
2019-10-22 21:22:23.581 +02:00 [Debug] Failed to authenticate HTTPS connection.
System.Security.Authentication.AuthenticationException: Authentication failed, see inner exception. ---> Interop+OpenSsl+SslException: SSL Handshake failed with OpenSSL error - SSL_ERROR_SSL. ---> Interop+Crypto+OpenSslCryptographicException: error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown
   --- End of inner exception stack trace ---
   at Interop.OpenSsl.DoSslHandshake(SafeSslHandle context, Byte[] recvBuf, Int32 recvOffset, Int32 recvCount, Byte[]& sendBuf, Int32& sendCount)
   at System.Net.Security.SslStreamPal.HandshakeInternal(SafeFreeCredentials credential, SafeDeleteContext& context, SecurityBuffer inputBuffer, SecurityBuffer outputBuffer, SslAuthenticationOptions sslAuthenticationOptions)
   --- End of inner exception stack trace ---
   at System.Net.Security.SslState.StartSendAuthResetSignal(ProtocolToken message, AsyncProtocolRequest asyncRequest, ExceptionDispatchInfo exception)
   at System.Net.Security.SslState.CheckCompletionBeforeNextReceive(ProtocolToken message, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.StartSendBlob(Byte[] incoming, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.ProcessReceivedBlob(Byte[] buffer, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.StartReadFrame(Byte[] buffer, Int32 readBytes, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.PartialFrameCallback(AsyncProtocolRequest asyncRequest)
--- End of stack trace from previous location where exception was thrown ---
   at System.Net.Security.SslState.ThrowIfExceptional()
   at System.Net.Security.SslState.InternalEndProcessAuthentication(LazyAsyncResult lazyResult)
   at System.Net.Security.SslState.EndProcessAuthentication(IAsyncResult result)
   at System.Net.Security.SslStream.EndAuthenticateAsServer(IAsyncResult asyncResult)
   at System.Net.Security.SslStream.<>c.<AuthenticateAsServerAsync>b__51_1(IAsyncResult iar)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter.InnerOnConnectionAsync(ConnectionAdapterContext context)
2019-10-22 21:22:23.583 +02:00 [Debug] Connection id ""0HLQN4KKEHMFI"" disconnecting.
2019-10-22 21:22:23.583 +02:00 [Debug] Connection id ""0HLQN4KKEHMFI"" sending FIN because: ""The client closed the connection.""
2019-10-22 21:22:23.584 +02:00 [Debug] Connection id ""0HLQN4KKEHMFI"" stopped.
2019-10-22 21:22:23.591 +02:00 [Debug] Connection id ""0HLQN4KKEHMFJ"" started.
2019-10-22 21:22:23.601 +02:00 [Information] Request starting HTTP/1.1 GET https://localhost:5001/sockjs-node/828/u0mgs32j/websocket  
2019-10-22 21:22:23.604 +02:00 [Debug] The request has an origin header: '"https://localhost:5001"'.
2019-10-22 21:22:23.605 +02:00 [Information] CORS policy execution successful.
2019-10-22 21:22:23.607 +02:00 [Debug] AuthenticationScheme: "Cookies" was successfully authenticated.
2019-10-22 21:22:23.607 +02:00 [Debug] The request path "/sockjs-node/828/u0mgs32j/websocket" does not match a supported file type
2019-10-22 21:22:23.608 +02:00 [Debug] Request did not match any routes
2019-10-22 21:22:23.642 +02:00 [Debug] Connection id ""0HLQN4KKEHMFK"" started.
2019-10-22 21:22:23.652 +02:00 [Information] Request starting HTTP/1.1 GET https://localhost:5001/sockjs-node/988/c03rqufx/websocket  
2019-10-22 21:22:23.653 +02:00 [Debug] The request has an origin header: '"https://localhost:5001"'.
2019-10-22 21:22:23.653 +02:00 [Information] CORS policy execution successful.
2019-10-22 21:22:23.654 +02:00 [Debug] AuthenticationScheme: "Cookies" was successfully authenticated.
2019-10-22 21:22:23.654 +02:00 [Debug] The request path "/sockjs-node/988/c03rqufx/websocket" does not match a supported file type
2019-10-22 21:22:23.654 +02:00 [Debug] Request did not match any routes
2019-10-22 21:22:24.646 +02:00 [Information] Request starting HTTP/1.1 GET https://localhost:5001/api/auth/isauthenticated  
2019-10-22 21:22:24.647 +02:00 [Debug] AuthenticationScheme: "Cookies" was successfully authenticated.
2019-10-22 21:22:24.647 +02:00 [Debug] The request path "/api/auth/isauthenticated" does not match a supported file type
2019-10-22 21:22:24.648 +02:00 [Debug] Request successfully matched the route with name 'null' and template '"api/Auth/IsAuthenticated"'
2019-10-22 21:22:24.670 +02:00 [Information] Route matched with "{action = \"IsAuthenticated\", controller = \"Auth\", page = \"\"}". Executing controller action with signature "Microsoft.AspNetCore.Mvc.IActionResult IsAuthenticated()" on controller "RepositoryNotifier.Controllers.AuthController" ("RepositoryNotifier").
2019-10-22 21:22:24.671 +02:00 [Debug] Execution plan of "authorization" filters (in the following order): ["None"]
2019-10-22 21:22:24.671 +02:00 [Debug] Execution plan of "resource" filters (in the following order): ["Microsoft.AspNetCore.Mvc.ViewFeatures.Internal.SaveTempDataFilter"]
2019-10-22 21:22:24.672 +02:00 [Debug] Execution plan of "action" filters (in the following order): ["Microsoft.AspNetCore.Mvc.Internal.ControllerActionFilter (Order: -2147483648)", "Microsoft.AspNetCore.Mvc.ModelBinding.UnsupportedContentTypeFilter (Order: -3000)"]
2019-10-22 21:22:24.672 +02:00 [Debug] Execution plan of "exception" filters (in the following order): ["None"]
2019-10-22 21:22:24.672 +02:00 [Debug] Execution plan of "result" filters (in the following order): ["Microsoft.AspNetCore.Mvc.ViewFeatures.Internal.SaveTempDataFilter"]
2019-10-22 21:22:24.681 +02:00 [Information] Executing action method "RepositoryNotifier.Controllers.AuthController.IsAuthenticated (RepositoryNotifier)" - Validation state: Valid
2019-10-22 21:22:24.684 +02:00 [Information] Executed action method "RepositoryNotifier.Controllers.AuthController.IsAuthenticated (RepositoryNotifier)", returned result "Microsoft.AspNetCore.Mvc.OkResult" in 0.4516ms.
2019-10-22 21:22:24.689 +02:00 [Information] Executing HttpStatusCodeResult, setting HTTP status code 200
2019-10-22 21:22:24.692 +02:00 [Information] Executed action "RepositoryNotifier.Controllers.AuthController.IsAuthenticated (RepositoryNotifier)" in 17.4118ms
2019-10-22 21:22:24.693 +02:00 [Debug] Connection id ""0HLQN4KKEHMFD"" completed keep alive response.
2019-10-22 21:22:24.693 +02:00 [Information] Request finished in 47.478ms 200 
2019-10-22 21:22:24.729 +02:00 [Information] Request starting HTTP/1.1 GET https://localhost:5001/fontawesome-webfont.woff2?v=4.7.0  
2019-10-22 21:22:24.729 +02:00 [Debug] The request has an origin header: '"https://localhost:5001"'.
2019-10-22 21:22:24.729 +02:00 [Information] CORS policy execution successful.
2019-10-22 21:22:24.730 +02:00 [Debug] AuthenticationScheme: "Cookies" was successfully authenticated.
2019-10-22 21:22:24.730 +02:00 [Debug] The request path "/fontawesome-webfont.woff2" does not match an existing file
2019-10-22 21:22:24.731 +02:00 [Debug] Request successfully matched the route with name '"default"' and template '"{controller}/{action=Index}/{id?}"'
2019-10-22 21:22:24.731 +02:00 [Debug] No actions matched the current request. Route values: ["controller=fontawesome-webfont.woff2", "action=Index"]
2019-10-22 21:22:24.731 +02:00 [Information] Request starting HTTP/1.1 GET https://localhost:5001/api/auth/getcurrentuser  
2019-10-22 21:22:24.731 +02:00 [Debug] No actions matched the current request. Route values: ["controller=fontawesome-webfont.woff2", "action=Index"]
2019-10-22 21:22:24.732 +02:00 [Debug] Request did not match any routes
2019-10-22 21:22:24.732 +02:00 [Debug] AuthenticationScheme: "Cookies" was successfully authenticated.
2019-10-22 21:22:24.733 +02:00 [Debug] The request path "/api/auth/getcurrentuser" does not match a supported file type
2019-10-22 21:22:24.733 +02:00 [Debug] Request successfully matched the route with name 'null' and template '"api/Auth/GetCurrentUser"'
2019-10-22 21:22:24.736 +02:00 [Debug] Connection id ""0HLQN4KKEHMFD"" completed keep alive response.
2019-10-22 21:22:24.736 +02:00 [Information] Request finished in 7.1551ms 200 font/woff2; charset=UTF-8
2019-10-22 21:22:24.737 +02:00 [Information] Route matched with "{action = \"GetCurrentUser\", controller = \"Auth\", page = \"\"}". Executing controller action with signature "RepositoryNotifier.DTO.UserTO GetCurrentUser()" on controller "RepositoryNotifier.Controllers.AuthController" ("RepositoryNotifier").
2019-10-22 21:22:24.738 +02:00 [Debug] Execution plan of "authorization" filters (in the following order): ["None"]
2019-10-22 21:22:24.738 +02:00 [Debug] Execution plan of "resource" filters (in the following order): ["Microsoft.AspNetCore.Mvc.ViewFeatures.Internal.SaveTempDataFilter"]
2019-10-22 21:22:24.738 +02:00 [Debug] Execution plan of "action" filters (in the following order): ["Microsoft.AspNetCore.Mvc.Internal.ControllerActionFilter (Order: -2147483648)", "Microsoft.AspNetCore.Mvc.ModelBinding.UnsupportedContentTypeFilter (Order: -3000)"]
2019-10-22 21:22:24.739 +02:00 [Debug] Execution plan of "exception" filters (in the following order): ["None"]
2019-10-22 21:22:24.739 +02:00 [Debug] Execution plan of "result" filters (in the following order): ["Microsoft.AspNetCore.Mvc.ViewFeatures.Internal.SaveTempDataFilter"]
2019-10-22 21:22:24.740 +02:00 [Information] Executing action method "RepositoryNotifier.Controllers.AuthController.GetCurrentUser (RepositoryNotifier)" - Validation state: Valid
2019-10-22 21:22:24.743 +02:00 [Information] Executed action method "RepositoryNotifier.Controllers.AuthController.GetCurrentUser (RepositoryNotifier)", returned result "Microsoft.AspNetCore.Mvc.ObjectResult" in 2.2061ms.
2019-10-22 21:22:24.744 +02:00 [Information] Request starting HTTP/1.1 GET https://localhost:5001/sockjs-node/info?t=1571772144737  
2019-10-22 21:22:24.744 +02:00 [Information] Request starting HTTP/1.1 GET https://localhost:5001/sockjs-node/info?t=1571772144736  
2019-10-22 21:22:24.744 +02:00 [Debug] AuthenticationScheme: "Cookies" was successfully authenticated.
2019-10-22 21:22:24.744 +02:00 [Debug] AuthenticationScheme: "Cookies" was successfully authenticated.
2019-10-22 21:22:24.745 +02:00 [Debug] The request path "/sockjs-node/info" does not match a supported file type
2019-10-22 21:22:24.745 +02:00 [Debug] The request path "/sockjs-node/info" does not match a supported file type
2019-10-22 21:22:24.745 +02:00 [Debug] Request successfully matched the route with name '"default"' and template '"{controller}/{action=Index}/{id?}"'
2019-10-22 21:22:24.745 +02:00 [Debug] Request successfully matched the route with name '"default"' and template '"{controller}/{action=Index}/{id?}"'
2019-10-22 21:22:24.745 +02:00 [Debug] List of registered output formatters, in the following order: ["Microsoft.AspNetCore.Mvc.Formatters.HttpNoContentOutputFormatter", "Microsoft.AspNetCore.Mvc.Formatters.StringOutputFormatter", "Microsoft.AspNetCore.Mvc.Formatters.StreamOutputFormatter", "Microsoft.AspNetCore.Mvc.Formatters.JsonOutputFormatter"]
2019-10-22 21:22:24.745 +02:00 [Debug] No actions matched the current request. Route values: ["controller=sockjs-node", "action=info"]
2019-10-22 21:22:24.746 +02:00 [Debug] No actions matched the current request. Route values: ["controller=sockjs-node", "action=info"]
2019-10-22 21:22:24.746 +02:00 [Debug] No actions matched the current request. Route values: ["controller=sockjs-node", "action=info"]
2019-10-22 21:22:24.747 +02:00 [Debug] No actions matched the current request. Route values: ["controller=sockjs-node", "action=info"]
2019-10-22 21:22:24.747 +02:00 [Debug] Request did not match any routes
2019-10-22 21:22:24.747 +02:00 [Debug] Request did not match any routes
2019-10-22 21:22:24.748 +02:00 [Debug] Connection id ""0HLQN4KKEHMFD"" completed keep alive response.
2019-10-22 21:22:24.748 +02:00 [Debug] Connection id ""0HLQN4KKEHMF2"" completed keep alive response.
2019-10-22 21:22:24.749 +02:00 [Information] Request finished in 4.9717ms 200 application/json; charset=UTF-8
2019-10-22 21:22:24.749 +02:00 [Information] Request finished in 5.478ms 200 application/json; charset=UTF-8
2019-10-22 21:22:24.749 +02:00 [Debug] No information found on request to perform content negotiation.
2019-10-22 21:22:24.750 +02:00 [Debug] Attempting to select an output formatter without using a content type as no explicit content types were specified for the response.
2019-10-22 21:22:24.750 +02:00 [Debug] Attempting to select the first formatter in the output formatters list which can write the result.
2019-10-22 21:22:24.750 +02:00 [Debug] Selected output formatter '"Microsoft.AspNetCore.Mvc.Formatters.JsonOutputFormatter"' and content type '"application/json"' to write the response.
2019-10-22 21:22:24.751 +02:00 [Information] Executing ObjectResult, writing value of type '"RepositoryNotifier.DTO.UserTO"'.
2019-10-22 21:22:24.779 +02:00 [Debug] Connection id ""0HLQN4KKEHMFL"" started.
2019-10-22 21:22:24.786 +02:00 [Debug] Connection id ""0HLQN4KKEHMFL"" received FIN.
2019-10-22 21:22:24.787 +02:00 [Debug] Failed to authenticate HTTPS connection.
System.Security.Authentication.AuthenticationException: Authentication failed, see inner exception. ---> Interop+OpenSsl+SslException: SSL Handshake failed with OpenSSL error - SSL_ERROR_SSL. ---> Interop+Crypto+OpenSslCryptographicException: error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown
   --- End of inner exception stack trace ---
   at Interop.OpenSsl.DoSslHandshake(SafeSslHandle context, Byte[] recvBuf, Int32 recvOffset, Int32 recvCount, Byte[]& sendBuf, Int32& sendCount)
   at System.Net.Security.SslStreamPal.HandshakeInternal(SafeFreeCredentials credential, SafeDeleteContext& context, SecurityBuffer inputBuffer, SecurityBuffer outputBuffer, SslAuthenticationOptions sslAuthenticationOptions)
   --- End of inner exception stack trace ---
   at System.Net.Security.SslState.StartSendAuthResetSignal(ProtocolToken message, AsyncProtocolRequest asyncRequest, ExceptionDispatchInfo exception)
   at System.Net.Security.SslState.CheckCompletionBeforeNextReceive(ProtocolToken message, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.StartSendBlob(Byte[] incoming, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.ProcessReceivedBlob(Byte[] buffer, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.StartReadFrame(Byte[] buffer, Int32 readBytes, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.PartialFrameCallback(AsyncProtocolRequest asyncRequest)
--- End of stack trace from previous location where exception was thrown ---
   at System.Net.Security.SslState.ThrowIfExceptional()
   at System.Net.Security.SslState.InternalEndProcessAuthentication(LazyAsyncResult lazyResult)
   at System.Net.Security.SslState.EndProcessAuthentication(IAsyncResult result)
   at System.Net.Security.SslStream.EndAuthenticateAsServer(IAsyncResult asyncResult)
   at System.Net.Security.SslStream.<>c.<AuthenticateAsServerAsync>b__51_1(IAsyncResult iar)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter.InnerOnConnectionAsync(ConnectionAdapterContext context)
2019-10-22 21:22:24.791 +02:00 [Debug] Connection id ""0HLQN4KKEHMFL"" disconnecting.
2019-10-22 21:22:24.791 +02:00 [Debug] Connection id ""0HLQN4KKEHMFL"" sending FIN because: ""The client closed the connection.""
2019-10-22 21:22:24.792 +02:00 [Debug] Connection id ""0HLQN4KKEHMFL"" stopped.
2019-10-22 21:22:24.797 +02:00 [Debug] Connection id ""0HLQN4KKEHMFM"" started.
2019-10-22 21:22:24.798 +02:00 [Information] Executed action "RepositoryNotifier.Controllers.AuthController.GetCurrentUser (RepositoryNotifier)" in 58.5918ms
2019-10-22 21:22:24.799 +02:00 [Debug] Connection id ""0HLQN4KKEHMFG"" completed keep alive response.
2019-10-22 21:22:24.799 +02:00 [Information] Request finished in 67.3826ms 200 application/json; charset=utf-8
2019-10-22 21:22:24.803 +02:00 [Debug] Connection id ""0HLQN4KKEHMFM"" received FIN.
2019-10-22 21:22:24.804 +02:00 [Debug] Failed to authenticate HTTPS connection.
System.Security.Authentication.AuthenticationException: Authentication failed, see inner exception. ---> Interop+OpenSsl+SslException: SSL Handshake failed with OpenSSL error - SSL_ERROR_SSL. ---> Interop+Crypto+OpenSslCryptographicException: error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown
   --- End of inner exception stack trace ---
   at Interop.OpenSsl.DoSslHandshake(SafeSslHandle context, Byte[] recvBuf, Int32 recvOffset, Int32 recvCount, Byte[]& sendBuf, Int32& sendCount)
   at System.Net.Security.SslStreamPal.HandshakeInternal(SafeFreeCredentials credential, SafeDeleteContext& context, SecurityBuffer inputBuffer, SecurityBuffer outputBuffer, SslAuthenticationOptions sslAuthenticationOptions)
   --- End of inner exception stack trace ---
   at System.Net.Security.SslState.StartSendAuthResetSignal(ProtocolToken message, AsyncProtocolRequest asyncRequest, ExceptionDispatchInfo exception)
   at System.Net.Security.SslState.CheckCompletionBeforeNextReceive(ProtocolToken message, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.StartSendBlob(Byte[] incoming, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.ProcessReceivedBlob(Byte[] buffer, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.StartReadFrame(Byte[] buffer, Int32 readBytes, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslState.PartialFrameCallback(AsyncProtocolRequest asyncRequest)
--- End of stack trace from previous location where exception was thrown ---
   at System.Net.Security.SslState.ThrowIfExceptional()
   at System.Net.Security.SslState.InternalEndProcessAuthentication(LazyAsyncResult lazyResult)
   at System.Net.Security.SslState.EndProcessAuthentication(IAsyncResult result)
   at System.Net.Security.SslStream.EndAuthenticateAsServer(IAsyncResult asyncResult)
   at System.Net.Security.SslStream.<>c.<AuthenticateAsServerAsync>b__51_1(IAsyncResult iar)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Server.Kestrel.Https.Internal.HttpsConnectionAdapter.InnerOnConnectionAsync(ConnectionAdapterContext context)
2019-10-22 21:22:24.806 +02:00 [Debug] Connection id ""0HLQN4KKEHMFM"" disconnecting.
2019-10-22 21:22:24.806 +02:00 [Debug] Connection id ""0HLQN4KKEHMFM"" sending FIN because: ""The client closed the connection.""
2019-10-22 21:22:24.806 +02:00 [Debug] Connection id ""0HLQN4KKEHMFM"" stopped.
2019-10-22 21:22:24.812 +02:00 [Information] Request starting HTTP/1.1 GET https://localhost:5001/assets/favicon.ico  
2019-10-22 21:22:24.813 +02:00 [Debug] AuthenticationScheme: "Cookies" was successfully authenticated.
2019-10-22 21:22:24.813 +02:00 [Debug] The request path "/assets/favicon.ico" does not match an existing file
2019-10-22 21:22:24.813 +02:00 [Debug] Request successfully matched the route with name '"default"' and template '"{controller}/{action=Index}/{id?}"'
2019-10-22 21:22:24.813 +02:00 [Debug] No actions matched the current request. Route values: ["controller=assets", "action=favicon.ico"]
2019-10-22 21:22:24.814 +02:00 [Debug] No actions matched the current request. Route values: ["controller=assets", "action=favicon.ico"]
2019-10-22 21:22:24.814 +02:00 [Debug] Request did not match any routes
2019-10-22 21:22:24.814 +02:00 [Debug] Connection id ""0HLQN4KKEHMFN"" started.
2019-10-22 21:22:24.815 +02:00 [Debug] Connection id ""0HLQN4KKEHMFG"" completed keep alive response.
2019-10-22 21:22:24.816 +02:00 [Information] Request finished in 3.6235ms 200 image/vnd.microsoft.icon; charset=UTF-8
2019-10-22 21:22:24.823 +02:00 [Information] Request starting HTTP/1.1 GET https://localhost:5001/sockjs-node/802/110t2d5p/websocket  
2019-10-22 21:22:24.824 +02:00 [Debug] The request has an origin header: '"https://localhost:5001"'.
2019-10-22 21:22:24.824 +02:00 [Information] CORS policy execution successful.
2019-10-22 21:22:24.825 +02:00 [Debug] AuthenticationScheme: "Cookies" was successfully authenticated.
2019-10-22 21:22:24.825 +02:00 [Debug] The request path "/sockjs-node/802/110t2d5p/websocket" does not match a supported file type
2019-10-22 21:22:24.825 +02:00 [Debug] Request did not match any routes
2019-10-22 21:22:24.838 +02:00 [Debug] Connection id ""0HLQN4KKEHMFO"" started.
2019-10-22 21:22:24.847 +02:00 [Information] Request starting HTTP/1.1 GET https://localhost:5001/sockjs-node/511/yq1hqjwf/websocket  
2019-10-22 21:22:24.847 +02:00 [Debug] The request has an origin header: '"https://localhost:5001"'.
2019-10-22 21:22:24.848 +02:00 [Information] CORS policy execution successful.
2019-10-22 21:22:24.848 +02:00 [Debug] AuthenticationScheme: "Cookies" was successfully authenticated.
2019-10-22 21:22:24.848 +02:00 [Debug] The request path "/sockjs-node/511/yq1hqjwf/websocket" does not match a supported file type
2019-10-22 21:22:24.848 +02:00 [Debug] Request did not match any routes
2019-10-22 21:22:26.116 +02:00 [Information] Request starting HTTP/1.1 GET https://localhost:5001/api/auth/isauthenticated  
2019-10-22 21:22:26.117 +02:00 [Debug] AuthenticationScheme: "Cookies" was successfully authenticated.
2019-10-22 21:22:26.117 +02:00 [Debug] The request path "/api/auth/isauthenticated" does not match a supported file type
2019-10-22 21:22:26.117 +02:00 [Debug] Request successfully matched the route with name 'null' and template '"api/Auth/IsAuthenticated"'
2019-10-22 21:22:26.118 +02:00 [Information] Route matched with "{action = \"IsAuthenticated\", controller = \"Auth\", page = \"\"}". Executing controller action with signature "Microsoft.AspNetCore.Mvc.IActionResult IsAuthenticated()" on controller "RepositoryNotifier.Controllers.AuthController" ("RepositoryNotifier").
2019-10-22 21:22:26.118 +02:00 [Debug] Execution plan of "authorization" filters (in the following order): ["None"]
2019-10-22 21:22:26.119 +02:00 [Debug] Execution plan of "resource" filters (in the following order): ["Microsoft.AspNetCore.Mvc.ViewFeatures.Internal.SaveTempDataFilter"]
2019-10-22 21:22:26.119 +02:00 [Debug] Execution plan of "action" filters (in the following order): ["Microsoft.AspNetCore.Mvc.Internal.ControllerActionFilter (Order: -2147483648)", "Microsoft.AspNetCore.Mvc.ModelBinding.UnsupportedContentTypeFilter (Order: -3000)"]
2019-10-22 21:22:26.119 +02:00 [Debug] Execution plan of "exception" filters (in the following order): ["None"]
2019-10-22 21:22:26.120 +02:00 [Debug] Execution plan of "result" filters (in the following order): ["Microsoft.AspNetCore.Mvc.ViewFeatures.Internal.SaveTempDataFilter"]
2019-10-22 21:22:26.120 +02:00 [Information] Executing action method "RepositoryNotifier.Controllers.AuthController.IsAuthenticated (RepositoryNotifier)" - Validation state: Valid
2019-10-22 21:22:26.121 +02:00 [Information] Executed action method "RepositoryNotifier.Controllers.AuthController.IsAuthenticated (RepositoryNotifier)", returned result "Microsoft.AspNetCore.Mvc.OkResult" in 0.0488ms.
2019-10-22 21:22:26.121 +02:00 [Information] Executing HttpStatusCodeResult, setting HTTP status code 200
2019-10-22 21:22:26.121 +02:00 [Information] Executed action "RepositoryNotifier.Controllers.AuthController.IsAuthenticated (RepositoryNotifier)" in 1.386ms
2019-10-22 21:22:26.122 +02:00 [Debug] Connection id ""0HLQN4KKEHMFG"" completed keep alive response.
2019-10-22 21:22:26.122 +02:00 [Information] Request finished in 6.3962ms 200 
2019-10-22 21:22:26.154 +02:00 [Information] Request starting HTTP/1.1 GET https://localhost:5001/api/auth/getcurrentuser  
2019-10-22 21:22:26.154 +02:00 [Information] Request starting HTTP/1.1 GET https://localhost:5001/api/auth/getcurrentusersrepositories  
2019-10-22 21:22:26.154 +02:00 [Information] Request starting HTTP/1.1 GET https://localhost:5001/api/job/getcommonkeywords  
2019-10-22 21:22:26.154 +02:00 [Information] Request starting HTTP/1.1 GET https://localhost:5001/api/job/getalljobs  
2019-10-22 21:22:26.154 +02:00 [Information] Request starting HTTP/1.1 GET https://localhost:5001/api/frequency/getfrequencies  
2019-10-22 21:22:26.155 +02:00 [Debug] AuthenticationScheme: "Cookies" was successfully authenticated.
2019-10-22 21:22:26.155 +02:00 [Debug] AuthenticationScheme: "Cookies" was successfully authenticated.
2019-10-22 21:22:26.156 +02:00 [Debug] AuthenticationScheme: "Cookies" was successfully authenticated.
2019-10-22 21:22:26.156 +02:00 [Debug] AuthenticationScheme: "Cookies" was successfully authenticated.
2019-10-22 21:22:26.156 +02:00 [Debug] The request path "/api/job/getcommonkeywords" does not match a supported file type
2019-10-22 21:22:26.156 +02:00 [Debug] AuthenticationScheme: "Cookies" was successfully authenticated.
2019-10-22 21:22:26.156 +02:00 [Debug] The request path "/api/auth/getcurrentuser" does not match a supported file type
2019-10-22 21:22:26.157 +02:00 [Debug] The request path "/api/auth/getcurrentusersrepositories" does not match a supported file type
2019-10-22 21:22:26.157 +02:00 [Debug] The request path "/api/job/getalljobs" does not match a supported file type
2019-10-22 21:22:26.157 +02:00 [Debug] Request successfully matched the route with name 'null' and template '"api/Job/GetCommonKeywords"'
2019-10-22 21:22:26.157 +02:00 [Debug] The request path "/api/frequency/getfrequencies" does not match a supported file type
2019-10-22 21:22:26.157 +02:00 [Debug] Request successfully matched the route with name 'null' and template '"api/Auth/GetCurrentUser"'
2019-10-22 21:22:26.158 +02:00 [Debug] Request successfully matched the route with name 'null' and template '"api/Auth/GetCurrentUsersRepositories"'
2019-10-22 21:22:26.158 +02:00 [Debug] Request successfully matched the route with name 'null' and template '"api/Job/GetAllJobs"'
2019-10-22 21:22:26.159 +02:00 [Debug] Request successfully matched the route with name 'null' and template '"api/Frequency/GetFrequencies"'
2019-10-22 21:22:26.159 +02:00 [Information] Route matched with "{action = \"GetCurrentUser\", controller = \"Auth\", page = \"\"}". Executing controller action with signature "RepositoryNotifier.DTO.UserTO GetCurrentUser()" on controller "RepositoryNotifier.Controllers.AuthController" ("RepositoryNotifier").
2019-10-22 21:22:26.160 +02:00 [Debug] Execution plan of "authorization" filters (in the following order): ["None"]
2019-10-22 21:22:26.161 +02:00 [Debug] Execution plan of "resource" filters (in the following order): ["Microsoft.AspNetCore.Mvc.ViewFeatures.Internal.SaveTempDataFilter"]
2019-10-22 21:22:26.161 +02:00 [Debug] Execution plan of "action" filters (in the following order): ["Microsoft.AspNetCore.Mvc.Internal.ControllerActionFilter (Order: -2147483648)", "Microsoft.AspNetCore.Mvc.ModelBinding.UnsupportedContentTypeFilter (Order: -3000)"]
2019-10-22 21:22:26.162 +02:00 [Information] Route matched with "{action = \"GetFrequencies\", controller = \"Frequency\", page = \"\"}". Executing controller action with signature "System.Collections.Generic.IList`1[RepositoryNotifier.Persistence.Job.JobFrequency] GetFrequencies()" on controller "RepositoryNotifier.Controllers.FrequencyController" ("RepositoryNotifier").
2019-10-22 21:22:26.162 +02:00 [Debug] Execution plan of "exception" filters (in the following order): ["None"]
2019-10-22 21:22:26.162 +02:00 [Debug] Execution plan of "authorization" filters (in the following order): ["None"]
2019-10-22 21:22:26.163 +02:00 [Debug] Execution plan of "result" filters (in the following order): ["Microsoft.AspNetCore.Mvc.ViewFeatures.Internal.SaveTempDataFilter"]
2019-10-22 21:22:26.163 +02:00 [Debug] Execution plan of "resource" filters (in the following order): ["Microsoft.AspNetCore.Mvc.ViewFeatures.Internal.SaveTempDataFilter"]
2019-10-22 21:22:26.164 +02:00 [Information] Executing action method "RepositoryNotifier.Controllers.AuthController.GetCurrentUser (RepositoryNotifier)" - Validation state: Valid
2019-10-22 21:22:26.164 +02:00 [Debug] Execution plan of "action" filters (in the following order): ["Microsoft.AspNetCore.Mvc.Internal.ControllerActionFilter (Order: -2147483648)", "Microsoft.AspNetCore.Mvc.ModelBinding.UnsupportedContentTypeFilter (Order: -3000)"]
2019-10-22 21:22:26.165 +02:00 [Information] Route matched with "{action = \"GetCommonKeywords\", controller = \"Job\", page = \"\"}". Executing controller action with signature "Microsoft.AspNetCore.Mvc.IActionResult GetCommonKeywords()" on controller "RepositoryNotifier.Controllers.JobController" ("RepositoryNotifier").
2019-10-22 21:22:26.165 +02:00 [Information] Executed action method "RepositoryNotifier.Controllers.AuthController.GetCurrentUser (RepositoryNotifier)", returned result "Microsoft.AspNetCore.Mvc.ObjectResult" in 0.0392ms.
2019-10-22 21:22:26.165 +02:00 [Debug] Execution plan of "exception" filters (in the following order): ["None"]
2019-10-22 21:22:26.165 +02:00 [Information] Route matched with "{action = \"GetAllJobs\", controller = \"Job\", page = \"\"}". Executing controller action with signature "Microsoft.AspNetCore.Mvc.IActionResult GetAllJobs()" on controller "RepositoryNotifier.Controllers.JobController" ("RepositoryNotifier").
2019-10-22 21:22:26.165 +02:00 [Debug] Execution plan of "authorization" filters (in the following order): ["None"]
2019-10-22 21:22:26.166 +02:00 [Debug] List of registered output formatters, in the following order: ["Microsoft.AspNetCore.Mvc.Formatters.HttpNoContentOutputFormatter", "Microsoft.AspNetCore.Mvc.Formatters.StringOutputFormatter", "Microsoft.AspNetCore.Mvc.Formatters.StreamOutputFormatter", "Microsoft.AspNetCore.Mvc.Formatters.JsonOutputFormatter"]
2019-10-22 21:22:26.166 +02:00 [Debug] Execution plan of "result" filters (in the following order): ["Microsoft.AspNetCore.Mvc.ViewFeatures.Internal.SaveTempDataFilter"]
2019-10-22 21:22:26.166 +02:00 [Debug] Execution plan of "authorization" filters (in the following order): ["None"]
2019-10-22 21:22:26.167 +02:00 [Debug] No information found on request to perform content negotiation.
2019-10-22 21:22:26.167 +02:00 [Debug] Execution plan of "resource" filters (in the following order): ["Microsoft.AspNetCore.Mvc.ViewFeatures.Internal.SaveTempDataFilter"]
2019-10-22 21:22:26.167 +02:00 [Debug] Execution plan of "resource" filters (in the following order): ["Microsoft.AspNetCore.Mvc.ViewFeatures.Internal.SaveTempDataFilter"]
2019-10-22 21:22:26.168 +02:00 [Debug] Attempting to select an output formatter without using a content type as no explicit content types were specified for the response.
2019-10-22 21:22:26.168 +02:00 [Information] Executing action method "RepositoryNotifier.Controllers.FrequencyController.GetFrequencies (RepositoryNotifier)" - Validation state: Valid
2019-10-22 21:22:26.168 +02:00 [Debug] Execution plan of "action" filters (in the following order): ["Microsoft.AspNetCore.Mvc.Internal.ControllerActionFilter (Order: -2147483648)", "Microsoft.AspNetCore.Mvc.ModelBinding.UnsupportedContentTypeFilter (Order: -3000)"]
2019-10-22 21:22:26.168 +02:00 [Debug] Execution plan of "action" filters (in the following order): ["Microsoft.AspNetCore.Mvc.Internal.ControllerActionFilter (Order: -2147483648)", "Microsoft.AspNetCore.Mvc.ModelBinding.UnsupportedContentTypeFilter (Order: -3000)"]
2019-10-22 21:22:26.169 +02:00 [Debug] Execution plan of "exception" filters (in the following order): ["None"]
2019-10-22 21:22:26.169 +02:00 [Debug] Attempting to select the first formatter in the output formatters list which can write the result.
2019-10-22 21:22:26.169 +02:00 [Information] Executed action method "RepositoryNotifier.Controllers.FrequencyController.GetFrequencies (RepositoryNotifier)", returned result "Microsoft.AspNetCore.Mvc.ObjectResult" in 0.7011ms.
2019-10-22 21:22:26.169 +02:00 [Debug] Execution plan of "exception" filters (in the following order): ["None"]
2019-10-22 21:22:26.170 +02:00 [Debug] Execution plan of "result" filters (in the following order): ["Microsoft.AspNetCore.Mvc.ViewFeatures.Internal.SaveTempDataFilter"]
2019-10-22 21:22:26.170 +02:00 [Debug] Selected output formatter '"Microsoft.AspNetCore.Mvc.Formatters.JsonOutputFormatter"' and content type '"application/json"' to write the response.
2019-10-22 21:22:26.170 +02:00 [Debug] List of registered output formatters, in the following order: ["Microsoft.AspNetCore.Mvc.Formatters.HttpNoContentOutputFormatter", "Microsoft.AspNetCore.Mvc.Formatters.StringOutputFormatter", "Microsoft.AspNetCore.Mvc.Formatters.StreamOutputFormatter", "Microsoft.AspNetCore.Mvc.Formatters.JsonOutputFormatter"]
2019-10-22 21:22:26.170 +02:00 [Debug] Execution plan of "result" filters (in the following order): ["Microsoft.AspNetCore.Mvc.ViewFeatures.Internal.SaveTempDataFilter"]
2019-10-22 21:22:26.171 +02:00 [Information] Executing ObjectResult, writing value of type '"RepositoryNotifier.DTO.UserTO"'.
2019-10-22 21:22:26.171 +02:00 [Information] Executing action method "RepositoryNotifier.Controllers.JobController.GetCommonKeywords (RepositoryNotifier)" - Validation state: Valid
2019-10-22 21:22:26.172 +02:00 [Debug] No information found on request to perform content negotiation.
2019-10-22 21:22:26.172 +02:00 [Information] Executing action method "RepositoryNotifier.Controllers.JobController.GetAllJobs (RepositoryNotifier)" - Validation state: Valid
2019-10-22 21:22:26.172 +02:00 [Debug] Attempting to select an output formatter without using a content type as no explicit content types were specified for the response.
2019-10-22 21:22:26.172 +02:00 [Information] Executed action "RepositoryNotifier.Controllers.AuthController.GetCurrentUser (RepositoryNotifier)" in 8.6395ms
2019-10-22 21:22:26.173 +02:00 [Debug] Attempting to select the first formatter in the output formatters list which can write the result.
2019-10-22 21:22:26.173 +02:00 [Debug] Connection id ""0HLQN4KKEHMFG"" completed keep alive response.
2019-10-22 21:22:26.173 +02:00 [Debug] Selected output formatter '"Microsoft.AspNetCore.Mvc.Formatters.JsonOutputFormatter"' and content type '"application/json"' to write the response.
2019-10-22 21:22:26.173 +02:00 [Information] Request finished in 19.2766ms 200 application/json; charset=utf-8
2019-10-22 21:22:26.174 +02:00 [Information] Executing ObjectResult, writing value of type '"System.Collections.Generic.List`1[[RepositoryNotifier.Persistence.Job.JobFrequency, RepositoryNotifier, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]]"'.
2019-10-22 21:22:26.175 +02:00 [Information] Route matched with "{action = \"GetCurrentUsersRepositories\", controller = \"Auth\", page = \"\"}". Executing controller action with signature "System.Threading.Tasks.Task`1[System.Collections.Generic.IList`1[System.String]] GetCurrentUsersRepositories()" on controller "RepositoryNotifier.Controllers.AuthController" ("RepositoryNotifier").
2019-10-22 21:22:26.175 +02:00 [Debug] Execution plan of "authorization" filters (in the following order): ["None"]
2019-10-22 21:22:26.176 +02:00 [Debug] Execution plan of "resource" filters (in the following order): ["Microsoft.AspNetCore.Mvc.ViewFeatures.Internal.SaveTempDataFilter"]
2019-10-22 21:22:26.176 +02:00 [Debug] Execution plan of "action" filters (in the following order): ["Microsoft.AspNetCore.Mvc.Internal.ControllerActionFilter (Order: -2147483648)", "Microsoft.AspNetCore.Mvc.ModelBinding.UnsupportedContentTypeFilter (Order: -3000)"]
2019-10-22 21:22:26.176 +02:00 [Debug] Execution plan of "exception" filters (in the following order): ["None"]
2019-10-22 21:22:26.176 +02:00 [Debug] Execution plan of "result" filters (in the following order): ["Microsoft.AspNetCore.Mvc.ViewFeatures.Internal.SaveTempDataFilter"]
2019-10-22 21:22:26.177 +02:00 [Information] Executing action method "RepositoryNotifier.Controllers.AuthController.GetCurrentUsersRepositories (RepositoryNotifier)" - Validation state: Valid
2019-10-22 21:22:26.180 +02:00 [Information] Executed action "RepositoryNotifier.Controllers.FrequencyController.GetFrequencies (RepositoryNotifier)" in 12.324ms
2019-10-22 21:22:26.180 +02:00 [Debug] Connection id ""0HLQN4KKEHMFA"" completed keep alive response.
2019-10-22 21:22:26.180 +02:00 [Information] Request finished in 25.7216ms 200 application/json; charset=utf-8
2019-10-22 21:22:26.181 +02:00 [Information] Request starting HTTP/1.1 GET https://localhost:5001/api/frequency/getfrequencies  
2019-10-22 21:22:26.181 +02:00 [Debug] AuthenticationScheme: "Cookies" was successfully authenticated.
2019-10-22 21:22:26.181 +02:00 [Debug] The request path "/api/frequency/getfrequencies" does not match a supported file type
2019-10-22 21:22:26.182 +02:00 [Debug] Request successfully matched the route with name 'null' and template '"api/Frequency/GetFrequencies"'
2019-10-22 21:22:26.182 +02:00 [Information] Route matched with "{action = \"GetFrequencies\", controller = \"Frequency\", page = \"\"}". Executing controller action with signature "System.Collections.Generic.IList`1[RepositoryNotifier.Persistence.Job.JobFrequency] GetFrequencies()" on controller "RepositoryNotifier.Controllers.FrequencyController" ("RepositoryNotifier").
2019-10-22 21:22:26.183 +02:00 [Debug] Execution plan of "authorization" filters (in the following order): ["None"]
2019-10-22 21:22:26.183 +02:00 [Debug] Execution plan of "resource" filters (in the following order): ["Microsoft.AspNetCore.Mvc.ViewFeatures.Internal.SaveTempDataFilter"]
2019-10-22 21:22:26.183 +02:00 [Debug] Execution plan of "action" filters (in the following order): ["Microsoft.AspNetCore.Mvc.Internal.ControllerActionFilter (Order: -2147483648)", "Microsoft.AspNetCore.Mvc.ModelBinding.UnsupportedContentTypeFilter (Order: -3000)"]
2019-10-22 21:22:26.183 +02:00 [Debug] AuthenticationScheme: "Cookies" was successfully authenticated.
2019-10-22 21:22:26.183 +02:00 [Debug] Execution plan of "exception" filters (in the following order): ["None"]
2019-10-22 21:22:26.183 +02:00 [Debug] AuthenticationScheme: "GitHub" was successfully authenticated.
2019-10-22 21:22:26.184 +02:00 [Debug] Execution plan of "result" filters (in the following order): ["Microsoft.AspNetCore.Mvc.ViewFeatures.Internal.SaveTempDataFilter"]
2019-10-22 21:22:26.184 +02:00 [Information] Executing action method "RepositoryNotifier.Controllers.FrequencyController.GetFrequencies (RepositoryNotifier)" - Validation state: Valid
2019-10-22 21:22:26.184 +02:00 [Information] Executed action method "RepositoryNotifier.Controllers.FrequencyController.GetFrequencies (RepositoryNotifier)", returned result "Microsoft.AspNetCore.Mvc.ObjectResult" in 0.0109ms.
2019-10-22 21:22:26.185 +02:00 [Debug] List of registered output formatters, in the following order: ["Microsoft.AspNetCore.Mvc.Formatters.HttpNoContentOutputFormatter", "Microsoft.AspNetCore.Mvc.Formatters.StringOutputFormatter", "Microsoft.AspNetCore.Mvc.Formatters.StreamOutputFormatter", "Microsoft.AspNetCore.Mvc.Formatters.JsonOutputFormatter"]
2019-10-22 21:22:26.185 +02:00 [Debug] No information found on request to perform content negotiation.
2019-10-22 21:22:26.185 +02:00 [Debug] Attempting to select an output formatter without using a content type as no explicit content types were specified for the response.
2019-10-22 21:22:26.185 +02:00 [Debug] Attempting to select the first formatter in the output formatters list which can write the result.
2019-10-22 21:22:26.185 +02:00 [Debug] Selected output formatter '"Microsoft.AspNetCore.Mvc.Formatters.JsonOutputFormatter"' and content type '"application/json"' to write the response.
2019-10-22 21:22:26.185 +02:00 [Information] Executing ObjectResult, writing value of type '"System.Collections.Generic.List`1[[RepositoryNotifier.Persistence.Job.JobFrequency, RepositoryNotifier, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]]"'.
2019-10-22 21:22:26.186 +02:00 [Information] Executed action "RepositoryNotifier.Controllers.FrequencyController.GetFrequencies (RepositoryNotifier)" in 1.9797ms
2019-10-22 21:22:26.186 +02:00 [Debug] Connection id ""0HLQN4KKEHMFG"" completed keep alive response.
2019-10-22 21:22:26.186 +02:00 [Information] Request finished in 5.7868ms 200 application/json; charset=utf-8
2019-10-22 21:22:26.214 +02:00 [Information] Executed action method "RepositoryNotifier.Controllers.JobController.GetAllJobs (RepositoryNotifier)", returned result "Microsoft.AspNetCore.Mvc.OkObjectResult" in 41.7825ms.
2019-10-22 21:22:26.215 +02:00 [Debug] List of registered output formatters, in the following order: ["Microsoft.AspNetCore.Mvc.Formatters.HttpNoContentOutputFormatter", "Microsoft.AspNetCore.Mvc.Formatters.StringOutputFormatter", "Microsoft.AspNetCore.Mvc.Formatters.StreamOutputFormatter", "Microsoft.AspNetCore.Mvc.Formatters.JsonOutputFormatter"]
2019-10-22 21:22:26.215 +02:00 [Debug] No information found on request to perform content negotiation.
2019-10-22 21:22:26.216 +02:00 [Debug] Attempting to select an output formatter without using a content type as no explicit content types were specified for the response.
2019-10-22 21:22:26.216 +02:00 [Debug] Attempting to select the first formatter in the output formatters list which can write the result.
2019-10-22 21:22:26.216 +02:00 [Debug] Selected output formatter '"Microsoft.AspNetCore.Mvc.Formatters.JsonOutputFormatter"' and content type '"application/json"' to write the response.
2019-10-22 21:22:26.216 +02:00 [Information] Executing ObjectResult, writing value of type '"System.Collections.Generic.List`1[[RepositoryNotifier.Persistence.Job.Job, RepositoryNotifier, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]]"'.
2019-10-22 21:22:26.238 +02:00 [Information] Executed action method "RepositoryNotifier.Controllers.JobController.GetCommonKeywords (RepositoryNotifier)", returned result "Microsoft.AspNetCore.Mvc.OkObjectResult" in 65.5975ms.
2019-10-22 21:22:26.238 +02:00 [Information] Executed action "RepositoryNotifier.Controllers.JobController.GetAllJobs (RepositoryNotifier)" in 66.5425ms
2019-10-22 21:22:26.238 +02:00 [Debug] List of registered output formatters, in the following order: ["Microsoft.AspNetCore.Mvc.Formatters.HttpNoContentOutputFormatter", "Microsoft.AspNetCore.Mvc.Formatters.StringOutputFormatter", "Microsoft.AspNetCore.Mvc.Formatters.StreamOutputFormatter", "Microsoft.AspNetCore.Mvc.Formatters.JsonOutputFormatter"]
2019-10-22 21:22:26.239 +02:00 [Debug] Connection id ""0HLQN4KKEHMF2"" completed keep alive response.
2019-10-22 21:22:26.239 +02:00 [Debug] No information found on request to perform content negotiation.
2019-10-22 21:22:26.239 +02:00 [Information] Request finished in 85.1441ms 200 application/json; charset=utf-8
2019-10-22 21:22:26.240 +02:00 [Debug] Attempting to select an output formatter without using a content type as no explicit content types were specified for the response.
2019-10-22 21:22:26.240 +02:00 [Debug] Attempting to select the first formatter in the output formatters list which can write the result.
2019-10-22 21:22:26.240 +02:00 [Debug] Selected output formatter '"Microsoft.AspNetCore.Mvc.Formatters.JsonOutputFormatter"' and content type '"application/json"' to write the response.
2019-10-22 21:22:26.241 +02:00 [Information] Executing ObjectResult, writing value of type '"System.Collections.Generic.List`1[[System.String, System.Private.CoreLib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]]"'.
2019-10-22 21:22:26.241 +02:00 [Information] Executed action "RepositoryNotifier.Controllers.JobController.GetCommonKeywords (RepositoryNotifier)" in 70.4573ms
2019-10-22 21:22:26.241 +02:00 [Debug] Connection id ""0HLQN4KKEHMFF"" completed keep alive response.
2019-10-22 21:22:26.242 +02:00 [Information] Request finished in 87.2854ms 200 application/json; charset=utf-8
2019-10-22 21:22:26.242 +02:00 [Information] Request starting HTTP/1.1 GET https://localhost:5001/api/job/getcommonkeywords  
2019-10-22 21:22:26.243 +02:00 [Debug] AuthenticationScheme: "Cookies" was successfully authenticated.
2019-10-22 21:22:26.243 +02:00 [Debug] The request path "/api/job/getcommonkeywords" does not match a supported file type
2019-10-22 21:22:26.243 +02:00 [Debug] Request successfully matched the route with name 'null' and template '"api/Job/GetCommonKeywords"'
2019-10-22 21:22:26.244 +02:00 [Information] Route matched with "{action = \"GetCommonKeywords\", controller = \"Job\", page = \"\"}". Executing controller action with signature "Microsoft.AspNetCore.Mvc.IActionResult GetCommonKeywords()" on controller "RepositoryNotifier.Controllers.JobController" ("RepositoryNotifier").
2019-10-22 21:22:26.244 +02:00 [Debug] Execution plan of "authorization" filters (in the following order): ["None"]
2019-10-22 21:22:26.244 +02:00 [Debug] Execution plan of "resource" filters (in the following order): ["Microsoft.AspNetCore.Mvc.ViewFeatures.Internal.SaveTempDataFilter"]
2019-10-22 21:22:26.245 +02:00 [Debug] Execution plan of "action" filters (in the following order): ["Microsoft.AspNetCore.Mvc.Internal.ControllerActionFilter (Order: -2147483648)", "Microsoft.AspNetCore.Mvc.ModelBinding.UnsupportedContentTypeFilter (Order: -3000)"]
2019-10-22 21:22:26.245 +02:00 [Debug] Execution plan of "exception" filters (in the following order): ["None"]
2019-10-22 21:22:26.245 +02:00 [Debug] Execution plan of "result" filters (in the following order): ["Microsoft.AspNetCore.Mvc.ViewFeatures.Internal.SaveTempDataFilter"]
2019-10-22 21:22:26.245 +02:00 [Information] Executing action method "RepositoryNotifier.Controllers.JobController.GetCommonKeywords (RepositoryNotifier)" - Validation state: Valid
2019-10-22 21:22:26.250 +02:00 [Information] Request starting HTTP/1.1 GET https://localhost:5001/api/job/getjobresults?frequency=1  
2019-10-22 21:22:26.250 +02:00 [Debug] AuthenticationScheme: "Cookies" was successfully authenticated.
2019-10-22 21:22:26.251 +02:00 [Debug] The request path "/api/job/getjobresults" does not match a supported file type
2019-10-22 21:22:26.251 +02:00 [Debug] Request successfully matched the route with name 'null' and template '"api/Job/GetJobResults"'
2019-10-22 21:22:26.271 +02:00 [Information] Route matched with "{action = \"GetJobResults\", controller = \"Job\", page = \"\"}". Executing controller action with signature "Microsoft.AspNetCore.Mvc.IActionResult GetJobResults(RepositoryNotifier.Persistence.Job.JobFrequency)" on controller "RepositoryNotifier.Controllers.JobController" ("RepositoryNotifier").
2019-10-22 21:22:26.272 +02:00 [Debug] Execution plan of "authorization" filters (in the following order): ["None"]
2019-10-22 21:22:26.272 +02:00 [Debug] Execution plan of "resource" filters (in the following order): ["Microsoft.AspNetCore.Mvc.ViewFeatures.Internal.SaveTempDataFilter"]
2019-10-22 21:22:26.272 +02:00 [Debug] Execution plan of "action" filters (in the following order): ["Microsoft.AspNetCore.Mvc.Internal.ControllerActionFilter (Order: -2147483648)", "Microsoft.AspNetCore.Mvc.ModelBinding.UnsupportedContentTypeFilter (Order: -3000)"]
2019-10-22 21:22:26.273 +02:00 [Debug] Execution plan of "exception" filters (in the following order): ["None"]
2019-10-22 21:22:26.273 +02:00 [Debug] Execution plan of "result" filters (in the following order): ["Microsoft.AspNetCore.Mvc.ViewFeatures.Internal.SaveTempDataFilter"]
2019-10-22 21:22:26.280 +02:00 [Information] Executed action method "RepositoryNotifier.Controllers.JobController.GetCommonKeywords (RepositoryNotifier)", returned result "Microsoft.AspNetCore.Mvc.OkObjectResult" in 34.2186ms.
2019-10-22 21:22:26.280 +02:00 [Debug] List of registered output formatters, in the following order: ["Microsoft.AspNetCore.Mvc.Formatters.HttpNoContentOutputFormatter", "Microsoft.AspNetCore.Mvc.Formatters.StringOutputFormatter", "Microsoft.AspNetCore.Mvc.Formatters.StreamOutputFormatter", "Microsoft.AspNetCore.Mvc.Formatters.JsonOutputFormatter"]
2019-10-22 21:22:26.281 +02:00 [Debug] No information found on request to perform content negotiation.
2019-10-22 21:22:26.281 +02:00 [Debug] Attempting to select an output formatter without using a content type as no explicit content types were specified for the response.
2019-10-22 21:22:26.281 +02:00 [Debug] Attempting to select the first formatter in the output formatters list which can write the result.
2019-10-22 21:22:26.282 +02:00 [Debug] Selected output formatter '"Microsoft.AspNetCore.Mvc.Formatters.JsonOutputFormatter"' and content type '"application/json"' to write the response.
2019-10-22 21:22:26.282 +02:00 [Information] Executing ObjectResult, writing value of type '"System.Collections.Generic.List`1[[System.String, System.Private.CoreLib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]]"'.
2019-10-22 21:22:26.282 +02:00 [Debug] Attempting to bind parameter '"p_frequency"' of type '"RepositoryNotifier.Persistence.Job.JobFrequency"' ...
2019-10-22 21:22:26.282 +02:00 [Information] Executed action "RepositoryNotifier.Controllers.JobController.GetCommonKeywords (RepositoryNotifier)" in 36.9445ms
2019-10-22 21:22:26.283 +02:00 [Debug] Connection id ""0HLQN4KKEHMF2"" completed keep alive response.
2019-10-22 21:22:26.283 +02:00 [Information] Request finished in 40.3828ms 200 application/json; charset=utf-8
2019-10-22 21:22:26.284 +02:00 [Debug] Attempting to bind model of type '"RepositoryNotifier.Persistence.Job.JobFrequency"' using the name '"frequency"' in request data ...
2019-10-22 21:22:26.285 +02:00 [Debug] Done attempting to bind model of type '"RepositoryNotifier.Persistence.Job.JobFrequency"' using the name '"frequency"'.
2019-10-22 21:22:26.285 +02:00 [Debug] Done attempting to bind parameter '"p_frequency"' of type '"RepositoryNotifier.Persistence.Job.JobFrequency"'.
2019-10-22 21:22:26.292 +02:00 [Information] Executing action method "RepositoryNotifier.Controllers.JobController.GetJobResults (RepositoryNotifier)" - Validation state: Valid
2019-10-22 21:22:26.429 +02:00 [Information] Executed action method "RepositoryNotifier.Controllers.JobController.GetJobResults (RepositoryNotifier)", returned result "Microsoft.AspNetCore.Mvc.OkResult" in 137.1529ms.
2019-10-22 21:22:26.430 +02:00 [Information] Executing HttpStatusCodeResult, setting HTTP status code 200
2019-10-22 21:22:26.430 +02:00 [Information] Executed action "RepositoryNotifier.Controllers.JobController.GetJobResults (RepositoryNotifier)" in 156.4151ms
2019-10-22 21:22:26.430 +02:00 [Debug] Connection id ""0HLQN4KKEHMFF"" completed keep alive response.
2019-10-22 21:22:26.430 +02:00 [Information] Request finished in 180.4855ms 200 
2019-10-22 21:22:26.572 +02:00 [Information] Executed action "RepositoryNotifier.Controllers.AuthController.GetCurrentUsersRepositories (RepositoryNotifier)" in 395.6894ms
2019-10-22 21:22:26.574 +02:00 [Error] An unhandled exception has occurred while executing the request.
System.AggregateException: One or more errors occurred. (Bad credentials) ---> Octokit.AuthorizationException: Bad credentials
   at Octokit.Connection.HandleErrors(IResponse response) in C:\projects\octokit-net\Octokit\Http\Connection.cs:line 665
   at Octokit.Connection.RunRequest(IRequest request, CancellationToken cancellationToken) in C:\projects\octokit-net\Octokit\Http\Connection.cs:line 642
   at Octokit.Connection.Run[T](IRequest request, CancellationToken cancellationToken) in C:\projects\octokit-net\Octokit\Http\Connection.cs:line 626
   at Octokit.ApiConnection.GetPage[TU](Uri uri, IDictionary`2 parameters, String accepts, ApiOptions options)
   at Octokit.ApiConnection.<>c__DisplayClass17_0`1.<<GetAll>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Octokit.ApiPagination.GetAllPages[T](Func`1 getFirstPage, Uri uri)
   at RepositoryNotifier.Service.Github.GithubApiService.GetGithubUser() in /home/morrismorrison/Development/Projects/NET/RepositoryObserver/RepositoryNotifier/Service/Github/GithubApiService.cs:line 56
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at RepositoryNotifier.Controllers.AuthController.GetCurrentUsersRepositories() in /home/morrismorrison/Development/Projects/NET/RepositoryObserver/RepositoryNotifier/Controllers/AuthController.cs:line 62
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
---> (Inner Exception #0) Octokit.AuthorizationException: Bad credentials
   at Octokit.Connection.HandleErrors(IResponse response) in C:\projects\octokit-net\Octokit\Http\Connection.cs:line 665
   at Octokit.Connection.RunRequest(IRequest request, CancellationToken cancellationToken) in C:\projects\octokit-net\Octokit\Http\Connection.cs:line 642
   at Octokit.Connection.Run[T](IRequest request, CancellationToken cancellationToken) in C:\projects\octokit-net\Octokit\Http\Connection.cs:line 626
   at Octokit.ApiConnection.GetPage[TU](Uri uri, IDictionary`2 parameters, String accepts, ApiOptions options)
   at Octokit.ApiConnection.<>c__DisplayClass17_0`1.<<GetAll>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Octokit.ApiPagination.GetAllPages[T](Func`1 getFirstPage, Uri uri)
   at RepositoryNotifier.Service.Github.GithubApiService.GetGithubUser() in /home/morrismorrison/Development/Projects/NET/RepositoryObserver/RepositoryNotifier/Service/Github/GithubApiService.cs:line 56
{"message":"Bad credentials","documentation_url":"https://developer.github.com/v3"}<---

2019-10-22 21:22:26.616 +02:00 [Information] Request starting HTTP/1.1 GET https://localhost:5001/api/auth/getcurrentusersrepositories  
2019-10-22 21:22:26.617 +02:00 [Debug] AuthenticationScheme: "Cookies" was successfully authenticated.
2019-10-22 21:22:26.617 +02:00 [Debug] The request path "/api/auth/getcurrentusersrepositories" does not match a supported file type
2019-10-22 21:22:26.618 +02:00 [Debug] Request successfully matched the route with name 'null' and template '"api/Auth/GetCurrentUsersRepositories"'
2019-10-22 21:22:26.618 +02:00 [Information] Route matched with "{action = \"GetCurrentUsersRepositories\", controller = \"Auth\", page = \"\"}". Executing controller action with signature "System.Threading.Tasks.Task`1[System.Collections.Generic.IList`1[System.String]] GetCurrentUsersRepositories()" on controller "RepositoryNotifier.Controllers.AuthController" ("RepositoryNotifier").
2019-10-22 21:22:26.619 +02:00 [Debug] Execution plan of "authorization" filters (in the following order): ["None"]
2019-10-22 21:22:26.619 +02:00 [Debug] Execution plan of "resource" filters (in the following order): ["Microsoft.AspNetCore.Mvc.ViewFeatures.Internal.SaveTempDataFilter"]
2019-10-22 21:22:26.619 +02:00 [Debug] Execution plan of "action" filters (in the following order): ["Microsoft.AspNetCore.Mvc.Internal.ControllerActionFilter (Order: -2147483648)", "Microsoft.AspNetCore.Mvc.ModelBinding.UnsupportedContentTypeFilter (Order: -3000)"]
2019-10-22 21:22:26.620 +02:00 [Debug] Execution plan of "exception" filters (in the following order): ["None"]
2019-10-22 21:22:26.620 +02:00 [Debug] Execution plan of "result" filters (in the following order): ["Microsoft.AspNetCore.Mvc.ViewFeatures.Internal.SaveTempDataFilter"]
2019-10-22 21:22:26.620 +02:00 [Information] Executing action method "RepositoryNotifier.Controllers.AuthController.GetCurrentUsersRepositories (RepositoryNotifier)" - Validation state: Valid
2019-10-22 21:22:26.621 +02:00 [Debug] AuthenticationScheme: "Cookies" was successfully authenticated.
2019-10-22 21:22:26.621 +02:00 [Debug] AuthenticationScheme: "GitHub" was successfully authenticated.
2019-10-22 21:22:26.630 +02:00 [Debug] Connection id ""0HLQN4KKEHMFD"" completed keep alive response.
2019-10-22 21:22:26.631 +02:00 [Information] Request finished in 476.2536ms 500 text/html; charset=utf-8
2019-10-22 21:22:26.854 +02:00 [Information] Executed action "RepositoryNotifier.Controllers.AuthController.GetCurrentUsersRepositories (RepositoryNotifier)" in 233.7137ms
2019-10-22 21:22:26.855 +02:00 [Error] An unhandled exception has occurred while executing the request.
System.AggregateException: One or more errors occurred. (Bad credentials) ---> Octokit.AuthorizationException: Bad credentials
   at Octokit.Connection.HandleErrors(IResponse response) in C:\projects\octokit-net\Octokit\Http\Connection.cs:line 665
   at Octokit.Connection.RunRequest(IRequest request, CancellationToken cancellationToken) in C:\projects\octokit-net\Octokit\Http\Connection.cs:line 642
   at Octokit.Connection.Run[T](IRequest request, CancellationToken cancellationToken) in C:\projects\octokit-net\Octokit\Http\Connection.cs:line 626
   at Octokit.ApiConnection.GetPage[TU](Uri uri, IDictionary`2 parameters, String accepts, ApiOptions options)
   at Octokit.ApiConnection.<>c__DisplayClass17_0`1.<<GetAll>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Octokit.ApiPagination.GetAllPages[T](Func`1 getFirstPage, Uri uri)
   at RepositoryNotifier.Service.Github.GithubApiService.GetGithubUser() in /home/morrismorrison/Development/Projects/NET/RepositoryObserver/RepositoryNotifier/Service/Github/GithubApiService.cs:line 56
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at RepositoryNotifier.Controllers.AuthController.GetCurrentUsersRepositories() in /home/morrismorrison/Development/Projects/NET/RepositoryObserver/RepositoryNotifier/Controllers/AuthController.cs:line 62
   at lambda_method(Closure , Object )
   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
---> (Inner Exception #0) Octokit.AuthorizationException: Bad credentials
   at Octokit.Connection.HandleErrors(IResponse response) in C:\projects\octokit-net\Octokit\Http\Connection.cs:line 665
   at Octokit.Connection.RunRequest(IRequest request, CancellationToken cancellationToken) in C:\projects\octokit-net\Octokit\Http\Connection.cs:line 642
   at Octokit.Connection.Run[T](IRequest request, CancellationToken cancellationToken) in C:\projects\octokit-net\Octokit\Http\Connection.cs:line 626
   at Octokit.ApiConnection.GetPage[TU](Uri uri, IDictionary`2 parameters, String accepts, ApiOptions options)
   at Octokit.ApiConnection.<>c__DisplayClass17_0`1.<<GetAll>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Octokit.ApiPagination.GetAllPages[T](Func`1 getFirstPage, Uri uri)
   at RepositoryNotifier.Service.Github.GithubApiService.GetGithubUser() in /home/morrismorrison/Development/Projects/NET/RepositoryObserver/RepositoryNotifier/Service/Github/GithubApiService.cs:line 56
{"message":"Bad credentials","documentation_url":"https://developer.github.com/v3"}<---

2019-10-22 21:22:26.875 +02:00 [Debug] Connection id ""0HLQN4KKEHMFF"" completed keep alive response.
2019-10-22 21:22:26.875 +02:00 [Information] Request finished in 259.0663ms 500 text/html; charset=utf-8
2019-10-22 21:22:31.584 +02:00 [Debug] Connection id ""0HLQN4KKEHMFN"" received FIN.
2019-10-22 21:22:31.584 +02:00 [Debug] Connection id ""0HLQN4KKEHMFJ"" received FIN.
2019-10-22 21:22:31.584 +02:00 [Debug] Connection id ""0HLQN4KKEHMFO"" received FIN.
2019-10-22 21:22:31.584 +02:00 [Debug] Connection id ""0HLQN4KKEHMFK"" received FIN.
2019-10-22 21:22:31.584 +02:00 [Debug] Connection id ""0HLQN4KKEHMFN"" sending FIN because: ""The client closed the connection.""
2019-10-22 21:22:31.585 +02:00 [Debug] Connection id ""0HLQN4KKEHMFO"" sending FIN because: ""The client closed the connection.""
2019-10-22 21:22:31.585 +02:00 [Debug] Connection id ""0HLQN4KKEHMFK"" sending FIN because: ""The client closed the connection.""
2019-10-22 21:22:31.585 +02:00 [Debug] Connection id ""0HLQN4KKEHMFJ"" sending FIN because: ""The client closed the connection.""
2019-10-22 21:22:31.586 +02:00 [Debug] Connection id ""0HLQN4KKEHMFN"" disconnecting.
2019-10-22 21:22:31.586 +02:00 [Debug] Connection id ""0HLQN4KKEHMFO"" disconnecting.
2019-10-22 21:22:31.588 +02:00 [Debug] Connection id ""0HLQN4KKEHMFK"" disconnecting.
2019-10-22 21:22:31.588 +02:00 [Debug] Connection id ""0HLQN4KKEHMFJ"" disconnecting.
2019-10-22 21:22:31.650 +02:00 [Information] Request finished in 6826.3047ms 101 
2019-10-22 21:22:31.651 +02:00 [Debug] Connection id ""0HLQN4KKEHMFN"" stopped.
2019-10-22 21:22:31.661 +02:00 [Information] Request finished in 6813.6458ms 101 
2019-10-22 21:22:31.661 +02:00 [Information] Request finished in 8060.506ms 101 
2019-10-22 21:22:31.662 +02:00 [Debug] Connection id ""0HLQN4KKEHMFJ"" stopped.
2019-10-22 21:22:31.662 +02:00 [Debug] Connection id ""0HLQN4KKEHMFO"" stopped.
2019-10-22 21:22:31.685 +02:00 [Information] Request finished in 8032.9356ms 101 
2019-10-22 21:22:31.685 +02:00 [Debug] Connection id ""0HLQN4KKEHMFK"" stopped.
2019-10-22 21:22:38.828 +02:00 [Debug] Hosting shutdown
2019-10-22 21:22:38.838 +02:00 [Debug] Connection id ""0HLQN4KKEHMF2"" disconnecting.
2019-10-22 21:22:38.838 +02:00 [Debug] Connection id ""0HLQN4KKEHMFA"" disconnecting.
2019-10-22 21:22:38.838 +02:00 [Debug] Connection id ""0HLQN4KKEHMFF"" disconnecting.
2019-10-22 21:22:38.838 +02:00 [Debug] Connection id ""0HLQN4KKEHMFG"" disconnecting.
2019-10-22 21:22:38.838 +02:00 [Debug] Connection id ""0HLQN4KKEHMFE"" disconnecting.
2019-10-22 21:22:38.839 +02:00 [Debug] Connection id ""0HLQN4KKEHMFD"" disconnecting.
2019-10-22 21:22:38.839 +02:00 [Debug] Connection id ""0HLQN4KKEHMF2"" sending FIN because: ""The Socket transport's send loop completed gracefully.""
2019-10-22 21:22:38.839 +02:00 [Debug] Connection id ""0HLQN4KKEHMFA"" sending FIN because: ""The Socket transport's send loop completed gracefully.""
2019-10-22 21:22:38.840 +02:00 [Debug] Connection id ""0HLQN4KKEHMFF"" sending FIN because: ""The Socket transport's send loop completed gracefully.""
2019-10-22 21:22:38.840 +02:00 [Debug] Connection id ""0HLQN4KKEHMFG"" sending FIN because: ""The Socket transport's send loop completed gracefully.""
2019-10-22 21:22:38.840 +02:00 [Debug] Connection id ""0HLQN4KKEHMFD"" sending FIN because: ""The Socket transport's send loop completed gracefully.""
2019-10-22 21:22:38.840 +02:00 [Debug] Connection id ""0HLQN4KKEHMFE"" sending FIN because: ""The Socket transport's send loop completed gracefully.""
2019-10-22 21:22:38.842 +02:00 [Debug] Connection id ""0HLQN4KKEHMF2"" stopped.
2019-10-22 21:22:38.842 +02:00 [Debug] Connection id ""0HLQN4KKEHMFA"" stopped.
2019-10-22 21:22:38.842 +02:00 [Debug] Connection id ""0HLQN4KKEHMFD"" stopped.
2019-10-22 21:22:38.842 +02:00 [Debug] Connection id ""0HLQN4KKEHMFE"" stopped.
2019-10-22 21:22:38.842 +02:00 [Debug] Connection id ""0HLQN4KKEHMFF"" stopped.
2019-10-22 21:22:38.842 +02:00 [Debug] Connection id ""0HLQN4KKEHMFG"" stopped.
